\documentclass[12pt,letterpaper]{article}
%DIF LATEXDIFF DIFFERENCE FILE
%DIF DEL first_submission.tex   Thu Jun  1 23:54:10 2023
%DIF ADD main.tex               Thu Jun  8 23:15:54 2023

%Packages
% \usepackage{textcomp}
% \usepackage{latexsym}
% \usepackage{url}
% \usepackage{amssymb}
% \usepackage{amsmath}
% \usepackage{mathtools}
% \usepackage{bm}
% \usepackage{array}
% \usepackage[version=3]{mhchem}
% \usepackage{ifthen}
% \usepackage{amsthm}
% \usepackage{amstext}
% \usepackage{enumerate}
% \usepackage{dcolumn}

\usepackage{epsfig}
\usepackage{graphicx}
\usepackage{caption}
\usepackage{hyperref}
\usepackage{lineno}
\usepackage{pdflscape}
\usepackage{mathtools}
\usepackage[osf]{mathpazo}
\usepackage{fullpage}
\usepackage{float}
\usepackage{xr} %linking to supplementaries
%DIF 30d30
%DIF < \usepackage{setspace} % double spacing
%DIF -------
\externaldocument{supplementaries}

\pagenumbering{arabic}


%---------------------------------------------
%
%       START
%
%---------------------------------------------
%DIF PREAMBLE EXTENSION ADDED BY LATEXDIFF
%DIF UNDERLINE PREAMBLE %DIF PREAMBLE
\RequirePackage[normalem]{ulem} %DIF PREAMBLE
\RequirePackage{color}\definecolor{RED}{rgb}{1,0,0}\definecolor{BLUE}{rgb}{0,0,1} %DIF PREAMBLE
\providecommand{\DIFaddtex}[1]{{\protect\color{blue}\uwave{#1}}} %DIF PREAMBLE
\providecommand{\DIFdeltex}[1]{{\protect\color{red}\sout{#1}}}                      %DIF PREAMBLE
%DIF SAFE PREAMBLE %DIF PREAMBLE
\providecommand{\DIFaddbegin}{} %DIF PREAMBLE
\providecommand{\DIFaddend}{} %DIF PREAMBLE
\providecommand{\DIFdelbegin}{} %DIF PREAMBLE
\providecommand{\DIFdelend}{} %DIF PREAMBLE
\providecommand{\DIFmodbegin}{} %DIF PREAMBLE
\providecommand{\DIFmodend}{} %DIF PREAMBLE
%DIF FLOATSAFE PREAMBLE %DIF PREAMBLE
\providecommand{\DIFaddFL}[1]{\DIFadd{#1}} %DIF PREAMBLE
\providecommand{\DIFdelFL}[1]{\DIFdel{#1}} %DIF PREAMBLE
\providecommand{\DIFaddbeginFL}{} %DIF PREAMBLE
\providecommand{\DIFaddendFL}{} %DIF PREAMBLE
\providecommand{\DIFdelbeginFL}{} %DIF PREAMBLE
\providecommand{\DIFdelendFL}{} %DIF PREAMBLE
%DIF HYPERREF PREAMBLE %DIF PREAMBLE
\providecommand{\DIFadd}[1]{\texorpdfstring{\DIFaddtex{#1}}{#1}} %DIF PREAMBLE
\providecommand{\DIFdel}[1]{\texorpdfstring{\DIFdeltex{#1}}{}} %DIF PREAMBLE
\newcommand{\DIFscaledelfig}{0.5}
%DIF HIGHLIGHTGRAPHICS PREAMBLE %DIF PREAMBLE
\RequirePackage{settobox} %DIF PREAMBLE
\RequirePackage{letltxmacro} %DIF PREAMBLE
\newsavebox{\DIFdelgraphicsbox} %DIF PREAMBLE
\newlength{\DIFdelgraphicswidth} %DIF PREAMBLE
\newlength{\DIFdelgraphicsheight} %DIF PREAMBLE
% store original definition of \includegraphics %DIF PREAMBLE
\LetLtxMacro{\DIFOincludegraphics}{\includegraphics} %DIF PREAMBLE
\newcommand{\DIFaddincludegraphics}[2][]{{\color{blue}\fbox{\DIFOincludegraphics[#1]{#2}}}} %DIF PREAMBLE
\newcommand{\DIFdelincludegraphics}[2][]{% %DIF PREAMBLE
\sbox{\DIFdelgraphicsbox}{\DIFOincludegraphics[#1]{#2}}% %DIF PREAMBLE
\settoboxwidth{\DIFdelgraphicswidth}{\DIFdelgraphicsbox} %DIF PREAMBLE
\settoboxtotalheight{\DIFdelgraphicsheight}{\DIFdelgraphicsbox} %DIF PREAMBLE
\scalebox{\DIFscaledelfig}{% %DIF PREAMBLE
\parbox[b]{\DIFdelgraphicswidth}{\usebox{\DIFdelgraphicsbox}\\[-\baselineskip] \rule{\DIFdelgraphicswidth}{0em}}\llap{\resizebox{\DIFdelgraphicswidth}{\DIFdelgraphicsheight}{% %DIF PREAMBLE
\setlength{\unitlength}{\DIFdelgraphicswidth}% %DIF PREAMBLE
\begin{picture}(1,1)% %DIF PREAMBLE
\thicklines\linethickness{2pt} %DIF PREAMBLE
{\color[rgb]{1,0,0}\put(0,0){\framebox(1,1){}}}% %DIF PREAMBLE
{\color[rgb]{1,0,0}\put(0,0){\line( 1,1){1}}}% %DIF PREAMBLE
{\color[rgb]{1,0,0}\put(0,1){\line(1,-1){1}}}% %DIF PREAMBLE
\end{picture}% %DIF PREAMBLE
}\hspace*{3pt}}} %DIF PREAMBLE
} %DIF PREAMBLE
\LetLtxMacro{\DIFOaddbegin}{\DIFaddbegin} %DIF PREAMBLE
\LetLtxMacro{\DIFOaddend}{\DIFaddend} %DIF PREAMBLE
\LetLtxMacro{\DIFOdelbegin}{\DIFdelbegin} %DIF PREAMBLE
\LetLtxMacro{\DIFOdelend}{\DIFdelend} %DIF PREAMBLE
\DeclareRobustCommand{\DIFaddbegin}{\DIFOaddbegin \let\includegraphics\DIFaddincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFaddend}{\DIFOaddend \let\includegraphics\DIFOincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFdelbegin}{\DIFOdelbegin \let\includegraphics\DIFdelincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFdelend}{\DIFOaddend \let\includegraphics\DIFOincludegraphics} %DIF PREAMBLE
\LetLtxMacro{\DIFOaddbeginFL}{\DIFaddbeginFL} %DIF PREAMBLE
\LetLtxMacro{\DIFOaddendFL}{\DIFaddendFL} %DIF PREAMBLE
\LetLtxMacro{\DIFOdelbeginFL}{\DIFdelbeginFL} %DIF PREAMBLE
\LetLtxMacro{\DIFOdelendFL}{\DIFdelendFL} %DIF PREAMBLE
\DeclareRobustCommand{\DIFaddbeginFL}{\DIFOaddbeginFL \let\includegraphics\DIFaddincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFaddendFL}{\DIFOaddendFL \let\includegraphics\DIFOincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFdelbeginFL}{\DIFOdelbeginFL \let\includegraphics\DIFdelincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFdelendFL}{\DIFOaddendFL \let\includegraphics\DIFOincludegraphics} %DIF PREAMBLE
%DIF COLORLISTINGS PREAMBLE %DIF PREAMBLE
\RequirePackage{listings} %DIF PREAMBLE
\RequirePackage{color} %DIF PREAMBLE
\lstdefinelanguage{DIFcode}{ %DIF PREAMBLE
%DIF DIFCODE_UNDERLINE %DIF PREAMBLE
  moredelim=[il][\color{red}\sout]{\%DIF\ <\ }, %DIF PREAMBLE
  moredelim=[il][\color{blue}\uwave]{\%DIF\ >\ } %DIF PREAMBLE
} %DIF PREAMBLE
\lstdefinestyle{DIFverbatimstyle}{ %DIF PREAMBLE
	language=DIFcode, %DIF PREAMBLE
	basicstyle=\ttfamily, %DIF PREAMBLE
	columns=fullflexible, %DIF PREAMBLE
	keepspaces=true %DIF PREAMBLE
} %DIF PREAMBLE
\lstnewenvironment{DIFverbatim}{\lstset{style=DIFverbatimstyle}}{} %DIF PREAMBLE
\lstnewenvironment{DIFverbatim*}{\lstset{style=DIFverbatimstyle,showspaces=true}}{} %DIF PREAMBLE
%DIF END PREAMBLE EXTENSION ADDED BY LATEXDIFF

\begin{document}
%Running head
\begin{flushright}
Version dated: \today
\end{flushright}

\bigskip
\bigskip
\begin{center}

\noindent{\Large \bf Innovation and elaboration on the avian tree of life}
\bigskip

\noindent {\normalsize \sc
Thomas Guillerme$^{1,*}$, 
Jen A. Bright$^{2}$,
Christopher R. Cooney${^1}$,
Emma C. Hughes${^1}$,
%DIF <  Zoe K. Varley${^3}$,
\DIFaddbegin \DIFadd{Zoe K. Varley${^3}$,
}\DIFaddend Natalie Cooper$^{3,+}$,
Andrew P. Beckerman$^{1,+}$,
and Gavin H. Thomas$^{1,4,+}$}\\
\noindent {\small \it 
$^1$School of Biosciences, University of Sheffield, Sheffield, S10 2TN, United Kingdom.\\
$^2$School of Natural Science, University of Hull, Hull, HU6 7RX, United Kingdom.\\
$^3$Natural History Museum, Cromwell Road, London, SW7 5BD, United Kingdom.\\
$^4$Bird Group, Department of Life Sciences, the Natural History Museum at Tring, Tring, United Kingdom.\\}


\end{center}
\bigskip
\noindent{*\bf Corresponding author.} \textit{guillert@tcd.ie}\\ 
\vspace{1in}

%Line numbering
\modulolinenumbers[1]
\linenumbers

\DIFdelbegin %DIFDELCMD < \doublespacing
%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelend %---------------------------------------------
%
%       ABSTRACT
%
%---------------------------------------------


\noindent (Keywords: macroevolution, elaboration, innovation, \DIFdelbegin \DIFdel{variance-covariance matrices}\DIFdelend \DIFaddbegin \DIFadd{evolutionary rate matrix}\DIFaddend , line of least resistance, GLMM)\\

\section{Abstract}

Widely documented, megaevolutionary jumps in \DIFaddbegin \DIFadd{phenotypic }\DIFaddend diversity continue to perplex researchers because it remains unclear whether these dramatic changes can emerge from microevolutionary processes.
Here we \DIFdelbegin \DIFdel{reconcile major changes in phenotypic diversity at macroevolutionary scales with gradual microevolutionary change using new multi-trait models }\DIFdelend \DIFaddbegin \DIFadd{tackle this question using new approaches for modeling multivariate traits %DIF > @L_multi_traits
 }\DIFaddend to evaluate the magnitude and distribution of elaboration and innovation in the evolution of bird beaks.
We find that elaboration\DIFdelbegin \DIFdel{dominates at macroevolutionary }\DIFdelend \DIFaddbegin \DIFadd{, evolution along the major axis of phenotypic change, is common at both macro- and megaevolutionary }\DIFaddend scales whereas innovation\DIFdelbegin \DIFdel{becomes }\DIFdelend \DIFaddbegin \DIFadd{, evolution away from the major axis of phenotypic change, is }\DIFaddend more prominent at megaevolutionary scales.
Indeed, the major axis of phenotypic change among species \DIFaddbegin \DIFadd{beak shapes }\DIFaddend at megaevolutionary scales is an emergent property of innovation across clades.
Our analyses suggest that the reorientation of phenotypes via innovation is a ubiquitous route for divergence that can arise through gradual change alone, opening up new avenues for evolution to explore. %DIF > @L_reworded_abstract

\section{Introduction}

%DIF <  TG: add these citations down there
%DIF <  However, we frequently observe major shifts in phenotypes among lineages \cite{venditti2011multiple, pagel2022general, BurinWhales}, suggesting that species or clades may also evolve away from the line of least resistance \cite{renaud2006conserved}.
%DIF <  At the macroevolutionary scale, evolution along a common major axis has been termed elaboration whereas evolution away from that major axis is termed innovation \cite{endler2005animal}. 
\DIFaddbegin \DIFadd{Understanding patterns and constraints in the adaptive evolution of species traits is a major goal of evolutionary research. At macroevolutionary scales, renewed attention has been given to variation in the tempo and mode of trait evolution among lineages \mbox{%DIFAUXCMD
\cite{venditti2011multiple, khabbazian2016fast, castiglione2018, hansen2022, larouche2022mosaic, pagel2022general, navalon2022environmental, basu2023tempo}}\hskip0pt%DIFAUXCMD
.
This has led to growing recognition of the importance of major jumps or discontinuities in evolutionary rates as drivers of diversification of species traits \mbox{%DIFAUXCMD
\cite{cooney2017mega, smaers2021evolution, goswami2022}}\hskip0pt%DIFAUXCMD
.
For example, in both bird beaks and mammal skulls there are major jumps in the rate of trait evolution early in clade history as morphological diversity expands \mbox{%DIFAUXCMD
\cite{cooney2017mega, goswami2022} }\hskip0pt%DIFAUXCMD
with lower magnitude jumps towards the present.
Major jumps in species trait values are consistent with the concept of megaevolution, describing major discontinuities in phenotypes observed from the fossil record, first proposed by Simpson \mbox{%DIFAUXCMD
\cite{simpson1944tempo,simpson1953major}}\hskip0pt%DIFAUXCMD
.
}\DIFaddend 

\DIFdelbegin \DIFdel{Evolutionary theory predicts that over short, microevolutionary , timescalesphenotypic evolution will tend to follow evolutionary lines of genetic }\DIFdelend \DIFaddbegin \DIFadd{Megaevolutionary dynamics appear to contrast with theory based on microevolutionary processes.
Over microevolutionary timescales, phenotypic evolution is expected to follow lines of }\DIFaddend least resistance \cite{schluter1996adaptive}.
\DIFdelbegin \DIFdel{Over longer timescales, the evolutionary }\DIFdelend \DIFaddbegin \DIFadd{The term ‘}\DIFaddend line of least resistance\DIFdelbegin \DIFdel{is analogous to the major axis of phenotypic variation \mbox{%DIFAUXCMD
\cite{marroig2005size,renaud2006conserved,fasanelli2022allometry}}\hskip0pt%DIFAUXCMD
.
Numerous studies show }\DIFdelend \DIFaddbegin \DIFadd{’ has been used both specifically to describe phenotypic evolution along the axis of greatest genetic variation \mbox{%DIFAUXCMD
\cite{schluter1996adaptive}}\hskip0pt%DIFAUXCMD
, and more loosely to describe the direction of phenotypic evolution with greatest observable variation \mbox{%DIFAUXCMD
\cite{marroig2005size}}\hskip0pt%DIFAUXCMD
.
Regardless of the definition used, the microevolutionary line of least resistance represents the direction with the most potential for evolution because it contains the most variation on which selection can act \mbox{%DIFAUXCMD
\cite{hansen2016, Rhoda2022, jablonski2022evolvability}}\hskip0pt%DIFAUXCMD
.
This expectation is supported by numerous studies showing }\DIFaddend that phenotypic divergence is \DIFdelbegin \DIFdel{biased along the evolutionary }\DIFdelend \DIFaddbegin \DIFadd{often biased along (i.e. aligned with) the }\DIFaddend line of least resistance \DIFdelbegin \DIFdel{, and therefore reflects the major axis of phenotypic variation, over }\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{marroig2005size, bolstad2014, walter2018, mcglothlin2018adaptive, tsuboi2018, Rhoda2022, mongle2022}}\hskip0pt%DIFAUXCMD
.
These studies suggest that alignment of phenotypic divergence with the line of least resistance may be common over comparatively short (}\DIFaddend 1-2 million year\DIFaddbegin \DIFadd{) timescales.
More generally, the alignment of macroevolutionary divergence with the microevolutionary line of least resistance is expected to decline over time \mbox{%DIFAUXCMD
\cite{mcglothlin2018adaptive}}\hskip0pt%DIFAUXCMD
.
However, there is evidence, for example from Anolis lizards, that stability in the direction of phenotypic evolution can extend over }\DIFaddend timescales \DIFdelbegin \DIFdel{\mbox{%DIFAUXCMD
\cite{marroig2005size, mcglothlin2018adaptive, tsuboi2018}}\hskip0pt%DIFAUXCMD
.
Recent evidencesuggests that this stability can even extend over macroevolutionary timescales }\DIFdelend spanning tens of millions of years \cite{mcglothlin2018adaptive}.
\DIFdelbegin \DIFdel{However, while the evolutionary line of genetic }\DIFdelend \DIFaddbegin 

\DIFadd{The juxtaposition of microevolutionary predictions to macro- and mega-evolutionary observations is striking and raises the question of how phenotypic divergence accumulates across evolutionary scales.
In particular, how can megaevolutionary jumps emerge from gradual micro- or macro-evolutionary processes of phenotypic evolution implied by microevolutionary predictions? \mbox{%DIFAUXCMD
\cite{hansen2016,jablonski2022evolvability}
}\hskip0pt%DIFAUXCMD
The original descriptions of microevolutionary lines of }\DIFaddend least resistance explicitly \DIFdelbegin \DIFdel{reflects genetic constraints, the macroevolutionary }\DIFdelend \DIFaddbegin \DIFadd{refer to genetic constraints.
However, at the macroevolutionary scale the }\DIFaddend major axis of phenotypic variation is an emergent property of \DIFdelbegin \DIFdel{genomic }\DIFdelend \DIFaddbegin \DIFadd{broader set of genetic }\DIFaddend and developmental constraints interacting with selection on a moving adaptive landscape \DIFdelbegin \DIFdel{\mbox{%DIFAUXCMD
\cite{jones2004evolution,BurinWhales}}\hskip0pt%DIFAUXCMD
. The widespread evidence of major shifts in phenotypes across the tree of life \mbox{%DIFAUXCMD
\cite{venditti2011multiple,pagel2022general,khabbazian2016fast,cooney2017mega,smaers2021evolution,goswami2022} }\hskip0pt%DIFAUXCMD
suggests a conflict in the direction of phenotypic evolution across scales where macroevolutionary divergence away from the }\DIFdelend %DIF >  GT: Masahito suggested a whole paragraph on adaptive landscapes and Simpon-Lande-Arnold ideas. I'm not sure that will help but hopefully the expanded intro overall works better. Possibly just needs another sentence or two here but it would break the flow. more citations of relevant papers here (the first 7 or so papers from Gustavo's paper would be relevant.
\DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{BurinWhales,jones2004evolution}}\hskip0pt%DIFAUXCMD
. %DIF >  GT: More references needed? I'm not sure where the idea originates.
At this scale the major axis of phenotypic variation among species \mbox{%DIFAUXCMD
\cite{marroig2005size, fasanelli2022allometry} }\hskip0pt%DIFAUXCMD
can be estimated from a matrix of divergences in species mean phenotypic traits (e.g. the }\textbf{\DIFadd{D}} \DIFadd{matrix of \mbox{%DIFAUXCMD
\cite{BlowsHiggie2003, mcglothlin2018adaptive}}\hskip0pt%DIFAUXCMD
.
These matrices do not take into account phylogenetic relationships among species.
In contrast, the }\textbf{\DIFadd{R}} \DIFadd{matrix of \mbox{%DIFAUXCMD
\cite{Houle2017} }\hskip0pt%DIFAUXCMD
describes the rate of among-species divergence in species traits and explicitly incorporates phylogeny (also see the }\textbf{\DIFadd{B}} \DIFadd{matrix of \mbox{%DIFAUXCMD
\cite{Machado2020}}\hskip0pt%DIFAUXCMD
) %DIF > @L_Rmatrix.
We hereafter refer to this }\textbf{\DIFadd{R}} \DIFadd{matrix as the }\textit{\DIFadd{evolutionary rate matrix}}\DIFadd{. %DIF > TG: TODO: change throughout (that's the VCV matrix now)
The }\DIFaddend major axis of \DIFdelbegin \DIFdel{phenotypic variation results in megaevolutionary divergence leading to distinct phenotypes and /or directions of evolution.
Resolving this conflict requires an understanding of the balance between deep-time reorientation of phenotypic major axes of evolutionamong clades and the directions of evolution of species within clades.}\DIFdelend \DIFaddbegin \DIFadd{the evolutionary rate matrix is a phylogenetic line of least resistance applicable at macro- and mega-evolutionary scales that is analogous to the genetic line of least resistance at microevolutionary scales. %DIF > @L_Rmatrix_full
}\DIFaddend 

\DIFdelbegin \DIFdel{Endler }\textit{\DIFdel{et al}}%DIFAUXCMD
\DIFdel{. \mbox{%DIFAUXCMD
\cite{endler2005animal}}\hskip0pt%DIFAUXCMD
, writing }\DIFdelend \DIFaddbegin \DIFadd{By studying the phylogenetic line of least resistance and related properties of the evolutionary rate matrix \mbox{%DIFAUXCMD
\cite{caetano2017ratematrix, caetano2018estimating, slater2019hierarchy} }\hskip0pt%DIFAUXCMD
at different scales we aim to gain new insights into the macro- and mega-evolutionary trajectories that lead to the diversity of beak shapes among extant bird species and clades.
While there is no strict definition that separates macro- and mega-evolution, we define the macroevolutionary scale as species divergence within clades and the megaevolutionary scale as divergence of clades within larger clades. %DIF > @L_mega_macro_def
The diversity of bird beaks has been well-studied at broad scales, particularly in relation to dietary ecology \mbox{%DIFAUXCMD
\cite{olsen2017feeding, felice2019, bright2016shapes, navalon2019, pigot2020macroevolutionary}}\hskip0pt%DIFAUXCMD
.
Previous studies have assessed how variation in rates of evolution of beak shape has generated this diversity \mbox{%DIFAUXCMD
\cite{cooney2017mega,chira2018correlates}}\hskip0pt%DIFAUXCMD
, revealing a general pattern of phylogenetically clustered evolutionary rates interspersed with major shifts deep in avian evolutionary history and in isolated lineages.
Major shifts indicate lineages that occupy distinct areas of beak shape space relative to their close relatives.
For example, a major shift in beak shape arises in the early divergence of Strisores separating hummingbirds (Trochilidae) from swifts (Apodidae), perhaps marking changes in ontogenetic trajectories \mbox{%DIFAUXCMD
\cite{navalon2021} }\hskip0pt%DIFAUXCMD
as hummingbirds diverged from broad billed ancestors (e.g. the Eocene stem Trochilidae fossils }\textit{\DIFadd{Parargornis messelensis}} \DIFadd{and }\textit{\DIFadd{Eocypselus rowei}} \DIFadd{\mbox{%DIFAUXCMD
\cite{mayr2003new, ksepka2013fossil}}\hskip0pt%DIFAUXCMD
). %DIF > @L_strisores_example
By assessing the multivariate directions of the evolution of beak shapes of species and clades we aim to deepen our understanding of how diversity in morphological form arises at macro- and mega-evolutionary scales in birds.
To do this we use data on bird beaks derived from 3D scans of museum specimens for $>$8700 species ($\sim$85\% of all extant bird species) and generate evolutionary rate matrices describing the directions of multivariate beak shape evolution.
The phylogenetic breadth of the data allows us to compare directions of beak evolution of subclades (e.g. orders) within the class Aves (the megaevolutionary scale) and of species within subclades (the macroevolutionary scale).%DIF > @L_macromega_definition2
}

\DIFadd{Various approaches have been developed to measure properties of trait matrices \mbox{%DIFAUXCMD
\cite{hansen2008measuring, pavlicev2009measuring, watanabe2022}}\hskip0pt%DIFAUXCMD
, broadly focused on describing the size, shape, and orientation of trait divergence \mbox{%DIFAUXCMD
\cite{phillips1999}}\hskip0pt%DIFAUXCMD
.
Our specific focus is on divergence of species and clades along or away from the phylogenetic line of least resistance.
We adopt concepts developed }\DIFaddend in the specific context of signal evolution \DIFdelbegin \DIFdel{, describe two routes to phenotypic novelty that we suggest can help to illuminate this challenge}\DIFdelend \DIFaddbegin \DIFadd{to describe these alternative routes to divergence \mbox{%DIFAUXCMD
\cite{endler2005animal}}\hskip0pt%DIFAUXCMD
}\DIFaddend .
Specifically, Endler \textit{et al}. \DIFaddbegin \DIFadd{(2005) }\DIFaddend refer to evolution of new phenotypes along the same major axis of phenotypic variation as \textit{elaboration}.
This is akin to phenotypic divergence among species following lines of least resistance, and, by definition, \DIFdelbegin \DIFdel{would be }\DIFdelend \DIFaddbegin \DIFadd{is }\DIFaddend expected to be a major driver of phenotypic evolution.
In contrast, \DIFdelbegin \DIFdel{they }\DIFdelend \DIFaddbegin \DIFadd{Endler }\textit{\DIFadd{et al}}\DIFadd{. (2005) }\DIFaddend refer to evolution away from the major axis of phenotypic variation, either in random directions or along limited axes, as \textit{innovation} \cite{endler2005animal}.
Relative to elaboration, innovation \DIFdelbegin \DIFdel{may be }\DIFdelend \DIFaddbegin \DIFadd{is }\DIFaddend more likely to generate phenotypic novelty, but\DIFdelbegin \DIFdel{this need not be synonymous with being }\DIFdelend \DIFaddbegin \DIFadd{, if directions of phenotypic evolution are constrained, would not be expected to be }\DIFaddend a primary driver of phenotypic diversity \DIFdelbegin \DIFdel{at broad scales.
These core concepts can now be defined using robust multivariate statistics about trait covariance applied at multiple phylogenetic scales and provide a powerful theoretical framework with which to study }\DIFdelend \DIFaddbegin \DIFadd{among closely related species.
We apply these terms to i) species divergence within clades, and ii) clade divergence within larger clades (Fig. \ref{Fig:cheatsheet}).
Both innovation and elaboration could in principle occur at the }\DIFaddend macro- \DIFdelbegin \DIFdel{to mega-evolutionary divergence }\DIFdelend \DIFaddbegin \DIFadd{or mega-evolutionary scale.
We expect that elaboration is more important at macroevolutionary scales (species diverging within clades) and innovation more important at megaevolutionary scale (divergence among clades).
For our bird beak data, macroevolutionary scales refer to the beak shapes of species elaborating or innovating relative to the phylogenetic line of least resistance of their taxonomic order, and megaevolutionary scales refer to avian orders elaborating or innovating relative to the phylogenetic line of least resistance for the entire class Aves.
For clarity throughout the manuscript, we refer to elaboration and innovation as the general concept described above but use the following terms to distinguish macroevolutionary scale (species) and megaevolutionary scale (clade) elaboration and innovation: $elaboration_{species}$, $innovation_{species}$, $elaboration_{clade}$ }\DIFaddend and \DIFdelbegin \DIFdel{in particular to determine the relative importance of:
i) idiosyncratic species-level innovation against clade-level innovation (i.
e. reorientation of }\DIFdelend \DIFaddbegin \DIFadd{$innovation_{clade}$.
We describe the algebraic formulation of elaboration and innovation for species and clades in detail in the Methods section.
Using these novel statistical tools, we estimate }\DIFaddend the \DIFdelbegin \DIFdel{directions of phenotypic variation), and ii) evolution along versus away from major axes of phenotypic variation.
The contributions of these multivariate, evolutionary pathways are poorly-known despite their importance for understanding the origins of diversity from macro- to mega-evolutionary scales.
Here we estimate for the first time, using novel statistical methods, the }\DIFdelend relative contribution of elaboration and innovation to the remarkable global radiation and diversity of \DIFdelbegin \DIFdel{birds }\DIFdelend \DIFaddbegin \DIFadd{bird }\DIFaddend beaks.


\DIFaddbegin \begin{figure}[!htbp]
\centering
   \includegraphics[width=1\textwidth]{Figures/cheat_sheet.pdf}
\caption{\scriptsize{Relationships between key figures in the main text.
A) represents the two first principal components (PCs) of the eight dimensional trait space with all bird beak shapes represented as gray circles and the Tinamiformes as red circles.
The gray ellipse represents the overall phylogenetic major axis of beak variation in birds.
Species aligned or away from this axis are respectively elaborators or innovators relative to the class Aves.
The red ellipse and axis represents the phylogenetic major axis of beak variation in Tinamiformes only.
Species aligned or away from this axis are respectively elaborators and innovators relative to the Tinamiformes order.
B) In Figure 2, we measured elaboration and innovation at the macroevolutionary scale , i.e. the position of each species in terms of $elaboration_{species}$ and $innovation_{species}$ relative to their order's phylogenetic major axis of beak variation.
A species $elaboration_{species}$ score is the species projection onto their order's phylogenetic major axis of beak variation(i. e. their position on that axis) whereas their $innovation_{species}$ score is their rejection from it (i.e. their distance away from that axis).
The colours on the tips of the phylogeny correspond to the median $elaboration_{species}$ (orange gradient) and $innovation_{species}$ (blue gradient).
C) Figure 4, we measured elaboration and innovation at the megaevolutionary scale, i.e. $elaboration_{clade}$ and $innovation_{clade}$.
The ellipses are scaled and centered and represent the clade's phylogenetic major axis of beak variation relative to the overall phylogenetic major axis of beak variation with the relative length of the ellipse on each dimension represented on the barplots as variance on each dimension.
A clade's $elaboration_{clade}$ and $innovation_{clade}$ scores are measured by projecting the clade's phylogenetic major axis of beak variation onto the overall phylogenetic major axis of beak variation and measuring the projection (elaboration) and rejection (innovation) from this axis as described above.
The distribution elaboration and innovation scores are taken from the projection of the 4000 pairs of evolutionary rate matrices with each pair being the focal level, e.g. order,and the parent level, e.g. the whole bird phylogeny.}}
\label{Fig:cheatsheet}
\end{figure}
\bigskip

%DIF >  \noindent \textbf{Figure \ref{Fig:cheatsheet}}


\section{\DIFadd{Results and Discussion}}

\DIFaddend \subsection{\DIFdelbegin \DIFdel{Modelling }\DIFdelend \DIFaddbegin \DIFadd{Modeling }\DIFaddend nested trait \DIFdelbegin \DIFdel{covariances}\DIFdelend \DIFaddbegin \DIFadd{covariance}\DIFaddend }
Addressing the relative contributions of elaboration and innovation to the origins of biodiversity in deep-time requires\DIFaddbegin \DIFadd{: }\DIFaddend 1) large, multivariate datasets to allow exploration of trait covariances at different scales, 2) reliable and efficient computational methods to estimate the major axes of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak shape }\DIFaddend variation, and 3) a set of mathematical tools that can estimate degrees of elaboration and innovation at any scale.
To meet the first challenge we use an eight-dimensional beak shape morphospace based on a geometric \DIFdelbegin \DIFdel{, }\DIFdelend morphometric dataset of 8748 species of birds (described in \cite{hughes2022global}).
To meet the other two challenges we introduce a novel analytical pipeline for measuring elaboration and innovation at the \DIFdelbegin \DIFdel{species-level and clade-level (Fig.S\ref{Fig:cheat_sheet} and S}\DIFdelend \DIFaddbegin \DIFadd{macroevolutionary scale (species within orders) and megaevolutionary scale (orders within the class Aves; Fig.
 \ref{Fig:cheatsheet} and }\DIFaddend \ref{Fig:mcmcmcglmm}).
To estimate the major axis of \DIFdelbegin \DIFdel{phenotypic variation we fit }\DIFdelend \DIFaddbegin \DIFadd{beak shape variation we estimate the evolutionary rate matrix \mbox{%DIFAUXCMD
\cite{Houle2017,Machado2020} }\hskip0pt%DIFAUXCMD
by fitting }\DIFaddend Bayesian phylogenetic generalised linear mixed model (pGLMM) models with beak shape as an eight-dimensional response variable \DIFdelbegin \DIFdel{(\mbox{%DIFAUXCMD
\cite{MCMCglmm}}\hskip0pt%DIFAUXCMD
).
From the posterior distribution of the variance-covariance matrix at any scale, the }\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{MCMCglmm}}\hskip0pt%DIFAUXCMD
.
The }\DIFaddend major axis can be identified from the leading eigenvector of the \DIFdelbegin \DIFdel{variance-covariance matrix.
We can estimate levels of elaboration and innovation based on that major axis of phenotypic variation (see Methods)}\DIFdelend \DIFaddbegin \DIFadd{posterior distribution of evolutionary rate matrices}\DIFaddend .
We regard the major axis of \DIFdelbegin \DIFdel{phenotypic variation }\DIFdelend \DIFaddbegin \DIFadd{the evolutionary rate matrix }\DIFaddend as a deep-time analogue of the microevolutionary \textbf{G} matrix \DIFdelbegin \DIFdel{\mbox{%DIFAUXCMD
\cite{robinson2013quantifying} }\hskip0pt%DIFAUXCMD
}\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{steppan2002comparative, robinson2013quantifying} }\hskip0pt%DIFAUXCMD
}\DIFaddend and posit that it represents the \DIFaddbegin \DIFadd{among species }\DIFaddend line of least evolutionary resistance, capturing the effects of historical contingency on multivariate evolution.
\DIFdelbegin \DIFdel{The }\DIFdelend \DIFaddbegin \DIFadd{This }\DIFaddend major axis of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation can be defined at any phylogenetic scale (i.e. for all species in the phylogeny, or for all species within any clade).
\DIFaddbegin \DIFadd{We can then estimate elaboration and innovation based on that major axis: elaboration is defined as the position on the phylogenetic major axis of beak variation and innovation is defined as the distance from the phylogenetic major axis of beak variation (see Methods). %DIF > @L_ei_meth_precision.
}\DIFaddend To assess the roles of elaboration and innovation \DIFaddbegin \DIFadd{in beak shape evolution }\DIFaddend across scales we fit our pGLMMs as nested models in which we define \DIFdelbegin \DIFdel{variance-covariance }\DIFdelend \DIFaddbegin \DIFadd{evolutionary rate }\DIFaddend matrices, and therefore major axes of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak shape }\DIFaddend variation, for
i) the entire phylogeny of all species included in the data (hereafter the \DIFdelbegin \DIFdel{global }\DIFdelend \DIFaddbegin \DIFadd{class-wide }\DIFaddend phylogenetic level),
ii) all species within each of nine clades mapping approximately to super-orders (the super-order level), and
iii) all species within each of 27 clades mapping to taxonomic orders (the order level).
These nested partitions of multivariate trait space provide the basis for all subsequent quantitative estimates of elaboration and innovation \DIFdelbegin \DIFdel{at multiple }\DIFdelend \DIFaddbegin \DIFadd{across }\DIFaddend scales.

\subsection{\DIFdelbegin \DIFdel{Elaboration dominates species-level }\DIFdelend \DIFaddbegin \DIFadd{Macroevolution: elaboration is a common route to }\DIFaddend divergence \DIFaddbegin \DIFadd{among species}\DIFaddend }
We tested the expectation, derived from adaptive radiation theory \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\cite{schluter1996adaptive,mcglothlin2018adaptive,slater2019hierarchy} }\hskip0pt%DIFAUXCMD
%DIF > @L_cite_adaptive
}\DIFaddend , that species divergence is biased along \DIFdelbegin \DIFdel{evolutionary }\DIFdelend \DIFaddbegin \DIFadd{phylogenetic }\DIFaddend lines of least resistance by calculating species-specific measures of \DIFdelbegin \DIFdel{innovation and elaboration.
If microevolutionary }\DIFdelend \DIFaddbegin \DIFadd{$elaboration_{species}$ and $innovation_{species}$.
If }\DIFaddend lines of least resistance are stable \DIFaddbegin \DIFadd{among species }\DIFaddend then we expect that species \DIFdelbegin \DIFdel{fall on a conserved major axis (elaboration}\DIFdelend \DIFaddbegin \DIFadd{typically fall on, or close to, a conserved phylogenetic major axis of beak variation ($elaboration_{species}$}\DIFaddend ) rather than \DIFdelbegin \DIFdel{off the line (innovation}\DIFdelend \DIFaddbegin \DIFadd{away from the axis ($innovation_{species}$}\DIFaddend ).
To assess this we \DIFdelbegin \DIFdel{calculated the multilinear algebraic projection of each species phenotype onto three }\DIFdelend \DIFaddbegin \DIFadd{projected the beak shape data for each species onto %DIF > @L_linear_algebra_stuff @L_beak_shape
the }\DIFaddend major axes of \DIFdelbegin \DIFdel{phenotypic variation (global phylogenetic , }\DIFdelend \DIFaddbegin \DIFadd{evolutionary rate matrices of their orders (see supplementary materials \ref{Fig:phylogeny_supplement} for projection onto the major axes of beak shape variation from the class-wide phylogenetic major axis of beak variation or their }\DIFaddend super-order\DIFdelbegin \DIFdel{, and order}\DIFdelend \DIFaddbegin \DIFadd{’s phylogenetic major axis of beak variation}\DIFaddend ).
We found \DIFaddbegin \DIFadd{typically }\DIFaddend higher values of \DIFdelbegin \DIFdel{elaboration than innovation overall across all levels (Fig\ref{Fig:phylogeny}) }\DIFdelend \DIFaddbegin \DIFadd{$elaboration_{species}$ than $innovation_{species}$ (Fig. \ref{Fig:phylogeny}) %DIF > @L_rephrase_wefound
and strong clustering of both $elaboration_{species}$ }\DIFaddend and \DIFdelbegin \DIFdel{globally more elaboration and innovation at the global phylogenetic and super-order levels compared to the order level.
}\DIFdelend \DIFaddbegin \DIFadd{$innovation_{species}$ (median $elaboration_{species}$ for all orders = 0.114, 95\%CI = 0.004-0.485;  $elaboration_{species}$ Pagel's $\lambda$ = 0.888; median $innovation_{species}$ for all orders = 0.071, 95\%CI = 0.018-0.224); $innovation_{species}$ Pagel's $\lambda$ = 0.848).%DIF > @L_ei_species_values. 
%DIF >  GT: TODO: add code for reproducibility 
}


\begin{figure}[!htbp]
\centering
   \includegraphics[width=1\textwidth]{Figures/InnovElabTree_main_text_revision.pdf}
\caption{\DIFaddFL{Avian phylogeny (n = 8748 species) showing Euclidean distance of species to the centroid of beak space (branches, cividis scale), and distributions of species beak shape $elaboration_{species}$ (inner circle, orange scale), and $innovation_{species}$ (outer circle, blue scale).
Elaboration and innovation scores represent comparisons of species at the order level.
Additional comparisons to super-orders and at the class-wide phylogenetic level are shown in Fig. \ref{Fig:phylogeny_supplement} and for family and super-family comparisons of the order Passeriformes in Fig. \ref{fig_phylogeny_passeriformes}.
}}
\label{Fig:phylogeny}
\end{figure}
\bigskip

%DIF >  \noindent \textbf{Figure \ref{Fig:phylogeny}}

\DIFadd{$Innovation_{species}$ is more apparent within non-passerine, rather than passerine, clades.
Most notably we see higher levels of innovation in orders with hook-shaped beaks (Pstittaciformes, Falconiformes, Accipitriformes), and the Apodiformes (swifts and hummingbirds).
Clades with hooked beaks have highly integrated beaks and brain cases and strong allometric patterns of shape variation, particularly among raptorial clades \mbox{%DIFAUXCMD
\cite{bright2016shapes, bright2019multifactorial}}\hskip0pt%DIFAUXCMD
.
Apodiformes are also notable for having both high innovation and elaboration scores.
The presence of extremes of elaboration and innovation might reflect the deep divergence and distinct ecologies of the two families, Trochilidae (hummingbirds) and Apodidae (swifts), that constitute the order.
The major axes of beak divergence within the order follow a trend from swifts (short, wide-gaped beaks; aerial insectivores) to hummingbirds (long, narrow, and often curved beaks; primarily nectarivores) that is not present within either family.
%DIF >  GT: Maybe needs a bit more here, possibly linking from integration to expectations of evolvability. Also, need to work on the end of the paragraph to articulate why the family split might lead to odd E and I (something along the lines of major axis is not elaboration within the families but instead is a mega level innovation).
This phylogenetic major axis of beak variation is likely to be representative of innovation between clades, rather than a line of least resistance within a clade.
}

\DIFaddend Phylogenetic distributions of \DIFdelbegin \DIFdel{innovation and elaboration }\DIFdelend \DIFaddbegin \DIFadd{$elaboration_{species}$ and $innovation_{species}$ }\DIFaddend broadly hold whether comparisons are made at the order, super-order \DIFdelbegin \DIFdel{, or global }\DIFdelend \DIFaddbegin \DIFadd{($elaboration_{species}$ Pagel's $\lambda$ = 0.853; $innovation_{species}$ Pagel's $\lambda$ = 0.914), or class-wide }\DIFaddend phylogenetic level (\DIFdelbegin \DIFdel{Fig. \ref{Fig:phylogeny}) %DIF < {#fig_correlations}),%TODO: maybe add to supp?
 }\DIFdelend \DIFaddbegin \DIFadd{$elaboration_{species}$ Pagel's $\lambda$ = 0.935; $innovation_{species}$ Pagel’s $\lambda$ = 0.837; Fig. \ref{Fig:phylogeny_supplement} and \ref{Fig:figure_correlations}) %DIF > @L_correlation_figure_link)
}\DIFaddend and in further nested analysis of the order Passeriformes (Fig. \DIFdelbegin \DIFdel{S\ref{fig_ellipses_passeriformes} and S}\DIFdelend \DIFaddbegin \DIFadd{\ref{fig_ellipses_passeriformes} and }\DIFaddend \ref{fig_phylogeny_passeriformes}).
\DIFdelbegin \DIFdel{However, the }\DIFdelend \DIFaddbegin \DIFadd{The }\DIFaddend extent of both elaboration and innovation tends to reduce as we move from \DIFdelbegin \DIFdel{global }\DIFdelend \DIFaddbegin \DIFadd{class-wide }\DIFaddend phylogenetic to within-order comparisons.
\DIFdelbegin \DIFdel{Broadly, and not surprisingly, this trend reflects the fact that species are more similar to the major axis of phenotypic variation of their close relatives than to the major axis of phenotypic variation of all birds.
The global }\DIFdelend \DIFaddbegin \DIFadd{We regard within-order comparisons as the most informative level of comparison at the macroevolutionary scale
%DIF >  GT: I added this so that we are clear on our focal analysis. The second part of the sentence is taken directly from one of reviewer 1's comments. TG:TODO: add this to the response letter @@@
because comparisons of species level values derived from the main axis of variation at the level of whole Neornithes might reflect global constraints in beak shape or an axis of innovation among clades (as with the Apodiformes discussed above).
The class-wide }\DIFaddend phylogenetic major axis of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation aligns closely with the \DIFaddbegin \DIFadd{raw (non-phylogenetic) }\DIFaddend major axis in the trait space and primarily describes variation between short, deep, and wide beaks at one extreme, and long, shallow, and narrow beaks at the other.
Our results point to the possibility that within clades the \DIFaddbegin \DIFadd{phylogenetic }\DIFaddend major axes of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation follow different directions.

\DIFdelbegin \DIFdel{Overall we found that, as expected from microevolutionary theory, }\DIFdelend \DIFaddbegin \DIFadd{Collectively, our results suggest that }\DIFaddend elaboration is the \DIFdelbegin \DIFdel{dominant mode of }\DIFdelend \DIFaddbegin \DIFadd{most common route by which species beak shape evolves.
While $elaboration_{species}$ is the main mode of beak shape }\DIFaddend divergence at the \DIFdelbegin \DIFdel{species level }\DIFdelend \DIFaddbegin \DIFadd{macroevolutionary scale %DIF > @L_dominant
}\DIFaddend (Fig. \ref{Fig:phylogeny})\DIFdelbegin \DIFdel{.
We find no consistent evidence that innovation is either positively correlated , or trades-off, with elaboration. %DIF < (Fig. S{#fig_correlations}). %TODO: maybe add to supp? 
However, we observe various combinations of elaboration and innovation across birds. Examples of high innovation and}\DIFdelend \DIFaddbegin \DIFadd{, it is also commonly positively correlated with $innovation_{species}$ (Fig. \ref{fable_correlations} and S8). %DIF > @L_correlation_figure_link).
In 14}\DIFaddend /\DIFdelbegin \DIFdel{or elaboration are often clustered within clades .
For example, compared to the global phylogenetic major axis of phenotypic variation, the Trochilidae (hummingbirds) in the order Apodiformes, have long narrow beaks and display high levels of elaboration and low innovation.
In contrast, the Bucerotiformes (hornbills), with large and deep beaks display high levels of innovation and low elaboration.
Other clades such as the Apodidae (swifts), in the order Apodiformes, and the distantly-related but convergent Hirundinidae (swallows and martins) in the order Passeriformes, have exceptionally wide beaks relative to length and display high levels of both elaboration and innovation.
The dominance of elaboration }\DIFdelend \DIFaddbegin \DIFadd{27 clades we find significant positive correlations with non-significant but positive trends in a further 11 clades.
Only two clades (Falconiformes and Columbiformes) hint at a negative correlation but are clearly non-significant.
The apparent predominance of $elaboration_{species}$ }\DIFaddend when measured at the \DIFdelbegin \DIFdel{species-level }\DIFdelend \DIFaddbegin \DIFadd{macroevolutionary scale }\DIFaddend is consistent with the idea of evolution along lines of least \DIFdelbegin \DIFdel{evolutionary resistance .
The frequent phylogenetic clustering of species that elaborate, innovate, or both, highlights the likelihood that major axes of phenotypic variation are not stable among clades.
This leads us to ask; if most organisms evolve incrementally along the major axis of phenotypic variation then how can we explain the staggering }\DIFdelend \DIFaddbegin \DIFadd{resistance %DIF >  @L_genetic_lol
yet the lack of trade-off between $elaboration_{species}$ and $innovation_{species}$ further suggests that evolvability, which is measured by elaboration along a single axis, may in fact be better regarded as multi-dimensional.
Indeed, if divergence can arise simultaneously along multiple axes then this may help to resolve the paradox of apparent shifts in phenotype that have contributed to the }\DIFaddend diversity of species morphologies across the tree of life\DIFdelbegin \DIFdel{? 
}\DIFdelend \DIFaddbegin \DIFadd{.
}\DIFaddend 

\begin{figure}[!htbp]
\centering
   \DIFdelbeginFL %DIFDELCMD < \includegraphics[width=0.9\textwidth]{Figures/InnovElabTree_main_text_with_box.pdf}
%DIFDELCMD < %%%
\DIFdelendFL \DIFaddbeginFL \includegraphics[width=0.5\textwidth]{Figures/correlations_fable_orders.pdf}
\DIFaddendFL \caption{\DIFaddbeginFL \DIFaddFL{Posterior correlations between $elaboration_{species}$ and $innovation_{species}$ for each super-order}\DIFaddendFL . \DIFaddbeginFL \DIFaddFL{The dots, thick lines and dashed lines represent respectively the median, 50\% confidence interval (CI) and 95\% CI scores. 14 out of 27 of the orders have a clear posterior correlation between $elaboration_{species}$ and $innovation_{species}$ (i.e. 95\% CI does not overlap with 0), nine have a somewhat positive posterior correlation (i.e. 50\% CI does not overlap with 0) and four have no clear posterior correlation. Note that no order has a clear negative posterior correlation. This trend suggests that there is no tradeoff between $elaboration_{species}$ and $innovation_{species}$ and that both can be common routes for beak shape evolution at the macroevolutionary scale.
}\DIFaddendFL }
\DIFdelbeginFL %DIFDELCMD < \label{Fig:phylogeny}
%DIFDELCMD < %%%
\DIFdelendFL \DIFaddbeginFL \label{fable_correlations}
\DIFaddendFL \end{figure}
\DIFdelbegin %DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelend \bigskip

\DIFdelbegin %DIFDELCMD < \noindent %%%
\textbf{\DIFdel{Figure \ref{Fig:phylogeny}:}}
%DIFAUXCMD
\DIFdel{The avian phylogeny (n = 8748 species) coloured by species beak shape innovation and elaboration level.
Blue and yellow colours on the branches of the phylogeny highlight species with relatively low distances to the global centroid }\DIFdelend %DIF >  \noindent \textbf{Figure \ref{fable_correlations}}
\DIFaddbegin 


\subsection{\DIFadd{Megaevolution: multiple routes to innovation throughout avian evolutionary history}}
\DIFadd{If deep-time jumps in beak shape \mbox{%DIFAUXCMD
\cite{cooney2017mega} }\hskip0pt%DIFAUXCMD
are the result of innovation we would expect to see changes in the orientation }\DIFaddend of trait space \DIFdelbegin \DIFdel{whereas orange to red colours indicate high distance from the centroid.
Innovation and elaboration scores are represented concentrically in a nested fashion with the most external pairs of circles (innovation and elaboration bands) representing comparisons for each species at the order level, then at the super-order level, and finally at the global phylogenetic level.The boxplots at the bottom of the figure summarise the distribution of the median innovation and elaboration values for all levels (bottom left), showing that elaboration is more common than elaboration,
and the pooled median innovation and elaboration values for all three different levels (bottom right), showing that elaboration and innovation is more common at higher taxonomic levels than at the order level. We can observe an increase in the distance of species to the global centroid of trait space at deeper phylogenetic scales with greater distances generally composed of more elaboration than innovation.
A companion plot showing further nested structure within the Passeriformes is shown in Fig.
S\ref{fig_phylogeny_passeriformes}. 
}%DIFDELCMD < 

%DIFDELCMD < %%%
\subsection{\DIFdel{Multiple routes to innovation throughout avian evolutionary history}}
%DIFAUXCMD
\addtocounter{subsection}{-1}%DIFAUXCMD
%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdel{To investigate the conflict implied by our observations of species-level elaboration and previously reported deep-time jumps in phenotype}\DIFdelend \DIFaddbegin \DIFadd{of clades (i.e. orders or super-orders) relative to the phylogenetic major axis of beak variation of the whole class Aves.
To test this}\DIFaddend , we measured the \DIFdelbegin \DIFdel{elaboration and innovation of different clades.
To do this we translated the major axes of phenotypic }\DIFdelend \DIFaddbegin \DIFadd{$elaboration_{clade}$ and $innovation_{clade}$ by translating the phylogenetic major axis of beak }\DIFaddend variation of each clade's (i.e. super-orders and orders) \DIFdelbegin \DIFdel{phylogenetic variance-covariance }\DIFdelend \DIFaddbegin \DIFadd{evolutionary rate }\DIFaddend matrix onto the \DIFdelbegin \DIFdel{global }\DIFdelend \DIFaddbegin \DIFadd{class-wide }\DIFaddend phylogenetic major axis of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation so that they shared the same origin in the shapespace.
We then used \DIFdelbegin \DIFdel{multilinear }\DIFdelend \DIFaddbegin \DIFadd{linear }\DIFaddend algebra to project %DIF > @L_linear_algebra_stuff
a focal clade's \DIFaddbegin \DIFadd{phylogenetic }\DIFaddend major axis of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation onto the \DIFdelbegin \DIFdel{global }\DIFdelend \DIFaddbegin \DIFadd{class-wide }\DIFaddend phylogenetic major axis of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation (Fig. S\DIFdelbegin \DIFdel{\ref{Fig:cheat_sheet}}\DIFdelend \DIFaddbegin \DIFadd{\ref{Fig:cheatsheet}}\DIFaddend ).
We interpret the measured \DIFdelbegin \DIFdel{projection (distance onto }\DIFdelend \DIFaddbegin \DIFadd{linear algebraic projection (Euclidean distance along }\DIFaddend the phylogenetic major axis of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation) as the clade's \DIFdelbegin \DIFdel{elaboration }\DIFdelend \DIFaddbegin \DIFadd{$elaboration_{clade}$ }\DIFaddend score and the measured \DIFdelbegin \DIFdel{rejection (}\DIFdelend \DIFaddbegin \DIFadd{linear algebraic rejection (Euclidean }\DIFaddend distance from the \DIFaddbegin \DIFadd{phylogenetic }\DIFaddend major axis of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation) as the clade's \DIFdelbegin \DIFdel{innovation score. A high innovation }\DIFdelend \DIFaddbegin \DIFadd{$innovation_{clade}$ score. %DIF >  @L_euclidean.
A high $innovation_{clade}$ }\DIFaddend score indicates that the direction of evolution of a clade differs from the direction of its parent clade (e.g. the direction of evolution of an order differs from its parent super-order).
These scores were \DIFdelbegin \DIFdel{measured }\DIFdelend \DIFaddbegin \DIFadd{calculated }\DIFaddend for each of 4000 posterior pairs of clade vs.
\DIFdelbegin \DIFdel{phylogenetic variance-covariance }\DIFdelend \DIFaddbegin \DIFadd{evolutionary rate }\DIFaddend matrix (Fig. \ref{Fig:ellipses} boxplots).
We found that for most clades the \DIFaddbegin \DIFadd{phylogenetic }\DIFaddend major axis of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation is not aligned with the \DIFdelbegin \DIFdel{global }\DIFdelend \DIFaddbegin \DIFadd{class-wide }\DIFaddend phylogenetic major axis of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation (Fig. \ref{Fig:ellipses}).
Despite the limitations of representing eight-dimensional space in 2D, our plots of the average elliptical representation of the \DIFdelbegin \DIFdel{phenotypic variance-covariance }\DIFdelend \DIFaddbegin \DIFadd{evolutionary rate }\DIFaddend matrix for the \DIFdelbegin \DIFdel{global }\DIFdelend \DIFaddbegin \DIFadd{class-wide }\DIFaddend phylogenetic major axis of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation and each super-order \DIFdelbegin \DIFdel{(Fig. \ref{Fig:ellipses}a) }\DIFdelend and order (Fig. \ref{Fig:ellipses}b) highlight the striking variation in the orientation of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak shape }\DIFaddend major axes among clades (Fig. \ref{Fig:orthogonality}).
\DIFdelbegin \DIFdel{Remarkably, more }\DIFdelend \DIFaddbegin \DIFadd{More }\DIFaddend than half of the assessed clades (4/8 super-orders and 15/27 orders) displayed higher median \DIFdelbegin \DIFdel{innovation than median elaboration }\DIFdelend \DIFaddbegin \DIFadd{$innovation_{clade}$ than median $elaboration_{clade}$ }\DIFaddend scores implying that \DIFdelbegin \DIFdel{, contrary to microevolutionary theory \mbox{%DIFAUXCMD
\cite{schluter1996adaptive,marroig2005size}}\hskip0pt%DIFAUXCMD
, innovation }\DIFdelend \DIFaddbegin \DIFadd{$innovation_{clade}$ }\DIFaddend is a more common generator of \DIFdelbegin \DIFdel{morphological diversity than elaboration }\DIFdelend \DIFaddbegin \DIFadd{beak shape diversity than $elaboration_{clade}$ }\DIFaddend in deep-time (i.e evidence at the clade scale).
\DIFdelbegin \DIFdel{However, there is substantial variation in elaboration and innovation }\DIFdelend \DIFaddbegin 

\begin{figure}[!htbp]
\centering
   \includegraphics[width=0.8\textwidth]{Figures/figure_phylo_spiro_dark.pdf}
\caption{\scriptsize{$Elaboration_{clade}$ and $innovation_{clade}$ at the megaevolutionary scale for each order and super-order in the bird phylogeny.
The black and gray ellipses are the scaled average evolutionary rate matrices from the pGLMM models for respectively the clades (black) and the class-wide phylogeny (gray).
The ellipses are centered on the position of the clade in the shapespace.
The associated black bar plots represent the variance on each of the eight dimensions in the shapespace with the top gray bar plot representing the variance on each of the eight dimensions for the class-wide phylogeny.
The orange and blue distributions represent respectively the distribution of the $elaboration_{clade}$ (orange) and the $innovation_{clade}$ (blue) scores for each clade and the dots are the median $elaboration_{clade}$ or $innovation_{clade}$ and the solid and the dashed lines representing the 50\% and the 95\% confidence intervales of the distribution of the scores across the 4000 posterior samples.
The ticks on the y-axis always represent five arbitrary units of elaboration and innovation.
An alternative visualization of the same information is available in the supplementary materials in Fig. \ref{Fig:ellipses_rainbow} along with a companion plot showing further nested structure within the Passeriformes in Fig. \ref{fig_ellipses_passeriformes}.}
}
\label{Fig:ellipses}
\end{figure}
\bigskip

%DIF >  \noindent \textbf{Figure \ref{Fig:ellipses}:}


\begin{figure}[!htbp]
\centering
   \includegraphics[width=1\textwidth]{Figures/orthogonality_results.pdf}
\caption{\scriptsize{Amount of orthogonality of each clade's phylogenetic major axis of beak variation compared to their parent or parent's parent clade.
The amount of orthogonality is represented on the horizontal axis and scales from 0 (modulo of $0^\circ$) to 1 (modulo of $90^\circ$) with the background gray and dashed gray lines representing, respectively, an orthogonality of 0.5 (modulo of $45^\circ$) and 0.75 (modulo of $67.5^\circ$).
Dots represent the median orthogonality of each clade and the lines their 95\% confidence intervals (CI).
No super-order's phylogenetic major axis of beak variation is parallel to the class-wide phylogenetic major axis of beak variation (purple lines and circles) and 22/27 orders are on average (median) at least half orthogonal ($>$  $45^\circ$) to the class-wide phylogenetic major axis of beak variation.
These results are to be contrasted with the orientation of each order relative to their super-order where on average (median) only 14/27 are at least half orthogonal to the orientation of their super-orders’s phylogenetic major axis of beak variation.
We also indicate the number of species (n) and the standard deviation (sd) of the orientation of their phylogenetic major axis of beak variation over the 4000 variance-covariance posteriors (sd; expressed in degrees).
Note that noclades are clearly parallel to their parent’s or parent’s parent’s ellipses (i.e.
no 95\% CI includes 0).
For each clade we also measured the posterior probability of each clade's orientation being different from their parent's clade or the class-wide phylogenetic major axis of beak variation relative to their sample size and sd.
The stars represent the posterior probability of the clade's orientation being different from the comparison clade (*** = pp $> 0.99$; ** = pp $>0.95$; * = pp $> 0.9$; . = pp $> 0.8$).
Only 1/6 super-orders, 8/27 orders relative to class-wide phylogeny and 6/27 relative to their super-orders have a pp $> 0.99$.
This is the result of the variation in sample size (n) and standard variation (sd) among clades.
A companion plot showing further nested structure within the Passeriformes is shown in Fig. \ref{fig_orthogonality_passeriformes}.}}
\label{Fig:orthogonality}
\end{figure}
\bigskip

%DIF >  \noindent \textbf{Figure \ref{Fig:orthogonality}:}


\DIFadd{Substantial variation in $elaboration_{clade}$ and $innovation_{clade}$ arises }\DIFaddend among clades including \DIFdelbegin \DIFdel{relatively low elaboration and innovation }\DIFdelend \DIFaddbegin \DIFadd{clades with relatively low $elaboration_{clade}$ and $innovation_{clade}$}\DIFaddend (e.g. Cuculiformes; Fig. \ref{Fig:ellipses}), low \DIFdelbegin \DIFdel{elaboration and high innovation }\DIFdelend \DIFaddbegin \DIFadd{$elaboration_{clade}$ and high $innovation_{clade}$ }\DIFaddend (e.g. Galliformes; Fig. \ref{Fig:ellipses}), high \DIFdelbegin \DIFdel{elaboration and low innovation }\DIFdelend \DIFaddbegin \DIFadd{$elaboration_{clade}$ and low $innovation_{clade}$ }\DIFaddend (e.g. Coraciiformes; Fig. \ref{Fig:ellipses}), and both relatively high \DIFdelbegin \DIFdel{elaboration and innovation }\DIFdelend \DIFaddbegin \DIFadd{$elaboration_{clade}$ and $innovation_{clade}$ }\DIFaddend (e.g. Podicipediformes; Fig. \ref{Fig:ellipses}).
These patterns hold across scales including within super-orders and at finer scales within the order Passeriformes (Fig.
\DIFdelbegin \DIFdel{S}\DIFdelend \ref{fig_ellipses_passeriformes}).
\DIFaddbegin \DIFadd{Evolutionary theory predicts that lines of least resistance should break down with time since divergence \mbox{%DIFAUXCMD
\cite{schluter1996adaptive, marroig2005size, mcglothlin2018adaptive}}\hskip0pt%DIFAUXCMD
.
We find that the extent of both $innovation_{clade}$ and $elaboration_{clade}$ increase with time since divergence from the root of the tree.
Similarly, when measured at the macroevolutionary scale, $innovation_{species}$ and $elaboration_{species}$ increase with species age (see supplementary Fig. \ref{Fig:figure_ei_clade_through_time} and \ref{Fig:figure_ei_species_through_time}). %DIF > @L_ei_through_time_plots
These results are consistent with both divergence in phenotype that is proportional to time (e.g. as expected under a Brownian motion model) and with the idea that changes in phenotypic correlations are expected to change over longer periods of time, potentially in response to moving adaptive landscapes \mbox{%DIFAUXCMD
\cite{BurinWhales}}\hskip0pt%DIFAUXCMD
.
}

\DIFaddend Our observations of heterogeneity in the orientation of the \DIFaddbegin \DIFadd{phylogenetic }\DIFaddend major axis of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation among clades is further supported by consistent evidence for \DIFaddbegin \DIFadd{high amounts of }\DIFaddend orthogonality of clades relative to the \DIFdelbegin \DIFdel{global }\DIFdelend \DIFaddbegin \DIFadd{class-wide }\DIFaddend phylogenetic major axis of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation (Fig. \ref{Fig:ellipses}\DIFdelbegin \DIFdel{- }\DIFdelend \DIFaddbegin \DIFadd{; }\DIFaddend Fig. \ref{Fig:orthogonality}\DIFdelbegin \DIFdel{).
}\DIFdelend \DIFaddbegin \DIFadd{; note that these comparisons are based on the evolutionary rate matrix). %DIF > @L_orientation_phylo
}\DIFaddend The median angle of the \DIFaddbegin \DIFadd{phylogenetic }\DIFaddend major axis of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation for subclades approaches orthogonality, differing from the \DIFdelbegin \DIFdel{global }\DIFdelend \DIFaddbegin \DIFadd{class-wide }\DIFaddend phylogenetic major axis of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation by $68.14^\circ$ ($95$\% CI: $22.83^\circ$-$89.09^\circ$\DIFdelbegin \DIFdel{- }\DIFdelend \DIFaddbegin \DIFadd{; }\DIFaddend Fig. \ref{Fig:orthogonality}).
Comparisons of orientation among subclades (i.e. orders within super-orders) show similar differences (median = $47.14^\circ$; $95$\% CI: $13.64^\circ$-$87.62^\circ$\DIFdelbegin \DIFdel{- }\DIFdelend \DIFaddbegin \DIFadd{; }\DIFaddend Fig. \ref{Fig:orthogonality}), suggesting that reorientations in trait space are largely unconstrained at the megaevolutionary \DIFdelbegin \DIFdel{level }\DIFdelend \DIFaddbegin \DIFadd{scale }\DIFaddend and are no more likely to occur along any one axis than another. 
This differs from previous inference \DIFdelbegin \DIFdel{on }\DIFdelend \DIFaddbegin \DIFadd{from }\DIFaddend a subset of our data \DIFaddbegin \DIFadd{analysed  without incorporating phylogeny, }\DIFaddend that implied generally consistent and low dimensionality within clades \cite{cooney2017mega}.
This suggests that there is no \DIFdelbegin \DIFdel{deep-time }\DIFdelend \DIFaddbegin \DIFadd{megaevolutionary }\DIFaddend analogue of the genetic line of least resistance and \DIFdelbegin \DIFdel{that across }\DIFdelend \DIFaddbegin \DIFadd{instead that megaevolutionary shifts more likely reflect changes in the adaptive landscape.
Across }\DIFaddend scales there is remarkable flexibility in the routes to innovation, consistent with the idea that morphological divergence may be less constrained in deep-time than is \DIFdelbegin \DIFdel{often }\DIFdelend \DIFaddbegin \DIFadd{sometimes }\DIFaddend assumed \cite{venditti2011multiple}. %DIF >  @L_occasionaly 

\DIFdelbegin %DIFDELCMD < \begin{figure}[!htbp]
%DIFDELCMD < \centering
%DIFDELCMD <    \includegraphics[width=1\textwidth]{Figures/ellipses.pdf}
%DIFDELCMD < %%%
%DIFDELCMD < \caption{%
{%DIFAUXCMD
\DIFdelFL{.}}
%DIFAUXCMD
%DIFDELCMD < \label{Fig:ellipses}
%DIFDELCMD < \end{figure}
%DIFDELCMD < \bigskip
%DIFDELCMD < 

%DIFDELCMD < \noindent %%%
\textbf{\DIFdel{Figure \ref{Fig:ellipses}:}}
%DIFAUXCMD
\DIFdel{Panel A) shows ellipses representing the scaled average posterior variance-covariance response from the pGLMM models for each super-order (coloured ellipses) compared to the global (all birds) phylogenetic component of the models (grey ellipses).
We scaled the ellipses so the length of the major axis of phenotypic variation of the clade ellipse is the same length as that of the global phylogenetic major axis ellipse (in eight dimensions).
The first inset ellipse plot shows the positions of all super-order ellipses relative to the global phylogenetic ellipse.
Subsequent inset plots show the results for each super-order. 
Inset barplots show the proportion of variance associated with each of the eight PC axes in shape morphospace.
The inset boxplots correspond to the elaboration (E) and innovation (I) scores for all 4000 posterior samples.
The dots represent the median elaboration and innovation values while the thick and dashed lines represent the 50\% and 95\% confidence intervals respectively.
These scores were calculated on the unscaled ellipses resulting in different scales of elaboration and innovation for each plot.
Panel B) shows the results for each order.
A companion plot showing further nested structure within the Passeriformes is shown in Fig. S\ref{fig_ellipses_passeriformes}.
}%DIFDELCMD < 

%DIFDELCMD < \bigskip
%DIFDELCMD < 

%DIFDELCMD < \bigskip
%DIFDELCMD < 

%DIFDELCMD < \begin{figure}[!htbp]
%DIFDELCMD < \centering
%DIFDELCMD <    \includegraphics[width=0.9\textwidth]{Figures/orthogonality_results.pdf}
%DIFDELCMD < %%%
%DIFDELCMD < \caption{%
{%DIFAUXCMD
\DIFdelFL{.}}
%DIFAUXCMD
%DIFDELCMD < \label{Fig:orthogonality}
%DIFDELCMD < \end{figure}
%DIFDELCMD < 

%DIFDELCMD < \bigskip
%DIFDELCMD < 

%DIFDELCMD < \noindent %%%
\textbf{\DIFdel{Figure \ref{Fig:orthogonality}:}}
%DIFAUXCMD
\textbf{\DIFdel{Clade's orthogonality.}}
%DIFAUXCMD
\DIFdel{Orthogonality of each clade's major axis of phenotypic variation compared to their parent or parent's parent clade.
Orthogonality is represented on the horizontal axis and scales from 0 (modulo of $0^\circ$) to 1 (modulo of $90^\circ$) with the background grey and dashed grey lines representing, respectively, an orthogonality of 0.5 (modulo of $45^\circ$) and 0.75 (modulo of $67.5^\circ$).
Dots represent the median orthogonality of each clade and the lines their 95\% CI.
No super-order's major axis of phenotypic variation is parallel to the global phylogenetic major axis (red lines and circles) and 12/25 orders are at least half orthogonal to the major axis of their super-order (green lines and circles).
These results are even clearer when compared to the global phylogenetic major axis (green dashed lines and open circles) where only 3/25 are less than half orthogonal.
We also indicate the number of species (n) and the standard deviation (sd) of their major axes of phenotypic variation orientation over the 4000 variance-covariance posteriors (sd; expressed in degrees).
For each clade we also measured the posterior probability of each clade's orientation being different from their parent's clade or the global phylogenetic major axis relative to their sample size and sd.
The stars represent the posterior probability of the clade's orientation being different from the comparison clade. (*** = pp $> 0.99$; ** = pp $>0.95$; * = pp $> 0.9$; . = pp $> 0.8$).
Only two out of six super-orders; 6/25 orders relative to their super-order and 9/25 orders relative to the phylogeny have a pp $> 0.99$.
This is the result of the variation in sample size (n) and standard variation (sd) among clades. 
A companion plot showing further nested structure within the Passeriformes is shown in Fig. S\ref{fig_orthogonality_passeriformes}.
}%DIFDELCMD < 

%DIFDELCMD < \bigskip
%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdel{Evolutionary innovation }\DIFdelend \DIFaddbegin \DIFadd{Evolutionary $innovation_{clade}$ }\DIFaddend can arise in \DIFdelbegin \DIFdel{any direction }\DIFdelend \DIFaddbegin \DIFadd{many directions }\DIFaddend in trait space \DIFdelbegin \DIFdel{.
Although the global }\DIFdelend \DIFaddbegin \DIFadd{(see examples in supplementary Fig. \ref{Fig:mikado}). %DIF > @L_any_dir
Although the class-wide }\DIFaddend phylogenetic major axis of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation aligns closely with the first dimension of the raw \DIFaddbegin \DIFadd{beak }\DIFaddend shapespace (70.96\% of the \DIFdelbegin \DIFdel{global }\DIFdelend \DIFaddbegin \DIFadd{class-wide }\DIFaddend phylogenetic major axis of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation is aligned with \DIFdelbegin \DIFdel{PC1}\DIFdelend \DIFaddbegin \DIFadd{the first dimension of the shape space}\DIFaddend ; Fig. \ref{Fig:ellipses}), no super-order, and only seven of the 27 orders (Procellariiformes, Pelecaniformes, Charadriiformes, Coraciiformes, Psittaciformes and Passeriformes), are aligned mainly \DIFdelbegin \DIFdel{with }\DIFdelend PC1 (Fig. \ref{Fig:ellipses}-bar plots); \DIFdelbegin \DIFdel{innovation }\DIFdelend \DIFaddbegin \DIFadd{$innovation_{clade}$ }\DIFaddend dominates at this scale.
This pattern of orthogonality also holds for sub-orders and families within the Passeriformes where only half of the sub-orders (Meliphagoidea, Corvides and Passerida) and seven of the 23 families (Eurylaimides, Meliphagoidea, Petroicidea, Fringilidae, Aegithaloidea, Pyconotiae, Nectariniidae) align with the Passeriformes \DIFaddbegin \DIFadd{phylogenetic }\DIFaddend major axis of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation (Fig.
\DIFdelbegin \DIFdel{S}\DIFdelend \ref{fig_ellipses_passeriformes}).
In addition to the lack of alignment of clade \DIFdelbegin \DIFdel{major axes of phenotypic }\DIFdelend \DIFaddbegin \DIFadd{phylogenetic major axis of beak }\DIFaddend variation with the \DIFaddbegin \DIFadd{class-wide }\DIFaddend phylogenetic major axis \DIFaddbegin \DIFadd{of beak variation for birds}\DIFaddend , we also found that within clades, \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak shape }\DIFaddend variation is either highly constrained (varying almost entirely along a single axis; e.g. in Galliformes) or higher-dimensional than \DIFdelbegin \DIFdel{expected (}\DIFdelend \DIFaddbegin \DIFadd{in some orders or superorder than across the whole class Aves (for example, }\DIFaddend there is no clear dominant \DIFdelbegin \DIFdel{major axis}\DIFdelend \DIFaddbegin \DIFadd{phylogenetic major axis of beak variation}\DIFaddend ; e.g. in Podicipediformes) from the \DIFdelbegin \DIFdel{global }\DIFdelend \DIFaddbegin \DIFadd{class-wide }\DIFaddend phylogenetic major axis of \DIFdelbegin \DIFdel{phenotypic variation. }\DIFdelend \DIFaddbegin \DIFadd{beak variation. %DIF > @L_highdimensions
}\DIFaddend For example, Accipitriformes (hawks and allies) have \DIFaddbegin \DIFadd{phylogenetic }\DIFaddend major axes of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation that mainly align with the second dimension of the shapespace suggesting distinct directions of \DIFaddbegin \DIFadd{beak }\DIFaddend evolution for the clade relative to the \DIFdelbegin \DIFdel{global }\DIFdelend \DIFaddbegin \DIFadd{class-wide }\DIFaddend phylogenetic level but uniformity in their beak shape within the clade.
In contrast, the Podicipediformes (grebes) are highly variable across all dimensions suggesting that all components of the shapespace are necessary to describe their beaks.
These observations illustrate multiple routes to \DIFdelbegin \DIFdel{innovation }\DIFdelend \DIFaddbegin \DIFadd{$innovation_{clade}$ }\DIFaddend and imply that within clades\DIFdelbegin \DIFdel{species }\DIFdelend \DIFaddbegin \DIFadd{, species beak shapes }\DIFaddend can evolve in many directions (akin to Endler \textit{et al}.'s \cite{endler2005animal} random innovation) or follow a single direction (akin to Endler \textit{et al}.'s \cite{endler2005animal} specific innovation) and that reorientations of trait space can arise in any direction, in any lineage, and at any time throughout avian history.

\subsection{Species elaborate \DIFaddbegin \textit{\DIFadd{and}} \DIFadd{innovate}\DIFaddend , clades innovate} \DIFdelbegin %DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelend %DIF > TG: @L_TLDR_nuance
The disconnect between observations of reorientation of trait space at the \DIFdelbegin \DIFdel{clade level and a dominance of elaboration at the species level }\DIFdelend \DIFaddbegin \DIFadd{macroevolutionary scale, and an apparent predominance of $elaboration_{species}$ }\DIFaddend could be viewed as a paradox.
We \DIFdelbegin \DIFdel{formalise }\DIFdelend \DIFaddbegin \DIFadd{formalize }\DIFaddend this paradox by testing whether elaboration is more common at higher taxonomic levels \DIFaddbegin \DIFadd{(megaevolutionary scale) }\DIFaddend than at the species level \DIFaddbegin \DIFadd{(macroevolutionary scale)}\DIFaddend .
We compared the area under the curve of the scaled density of \DIFdelbegin \DIFdel{elaboration and innovation at both the species level and the clade level}\DIFdelend \DIFaddbegin \DIFadd{$elaboration_{species}$, $innovation_{species}$, $elaboration_{clade}$ and $innovation_{clade}$}\DIFaddend .
We then measured the difference between both areas and found that there was a significant difference in \DIFdelbegin \DIFdel{innovation among species and clades }\DIFdelend \DIFaddbegin \DIFadd{$innovation_{species}$ and $innovation_{clade}$ }\DIFaddend but no clear difference in \DIFdelbegin \DIFdel{elaboration }\DIFdelend \DIFaddbegin \DIFadd{$elaboration_{species}$ and $elaboration_{clade}$ }\DIFaddend (Fig. \ref{Fig:relative_EI}).
This difference confirms our observation that \DIFdelbegin \DIFdel{while innovation is common and perhaps even }\DIFdelend \DIFaddbegin \DIFadd{innovation is indeed common and }\DIFaddend dominant at the \DIFdelbegin \DIFdel{clade level, elaboration dominates at the species level}\DIFdelend \DIFaddbegin \DIFadd{megaevolutionary scale, however, the contributions of elaboration and innovation are shown to be indistinguishable at the macroevolutionary scale}\DIFaddend .
So, although species \DIFaddbegin \DIFadd{often }\DIFaddend evolve preferentially along a shared \DIFdelbegin \DIFdel{major axis of phenotypic variance, this major axis changes frequently }\DIFdelend \DIFaddbegin \DIFadd{phylogenetic major axes of beak variation within clades, there is frequent deviation from these major axes, and the phylogenetic major axes of beak variation change }\DIFaddend among clades.
\DIFdelbegin \DIFdel{However, we suggest that }\DIFdelend \DIFaddbegin \DIFadd{This could be explained in part because the evolutionary rate matrix (}\textbf{\DIFadd{R}}\DIFadd{) is more influenced by shifts in the adaptive landscape through time and thus reflecting natural selection in deeper nodes more than in younger nodes (Fig. \ref{Fig:figure_ei_clade_through_time} and \ref{Fig:figure_ei_species_through_time}). %DIF > @L_R_selection
}

\begin{figure}[!htbp]
\centering
   \includegraphics[width=1\textwidth]{Figures/relative_EI_gg_style_unscaled.pdf}
\caption{\DIFaddFL{Comparisons of relative $elaboration_{species}$ and $innovation_{species}$ (Fig. \ref{Fig:phylogeny}) and $elaboration_{clade}$ and $innovation_{clade}$ (Fig. \ref{Fig:ellipses}).
a) the $elaboration_{species}$ (purple) for each species in each 4000 posterior samples compared to the $elaboration_{clade}$ for each of the 35 clades (yellow).
The $elaboration_{clade}$ scores are scaled by the maximum score within the clade (corresponding to the scaled ellipses in Fig. \ref{Fig:ellipses}) and the scores for species are scaled by the maximum $elaboration_{species}$ score.
This allows us to compare $elaboration_{species}$ in and $elaboration_{clade}$ despite their different sample sizes.
We use the Bhattacharryya Coefficient to quantify the overlap of the area under the curve for species and clades.
For elaboration the overlap is Bhattacharryya Coefficient = 0.217, indicating no distinguishable dissimilarity in the amount of $elaboration_{species}$ and $elaboration_{clade}$ across the posterior samples.
b) as a) but using $innovation_{species}$ and $innovation_{clade}$ scores.
For innovation the overlap has Bhattacharryya Coefficient = 0.039, which indicates a clear difference in the amount of $innovation_{species}$ and $innovation_{clade}$ across the posterior samples.}}
\label{Fig:relative_EI}
\end{figure}
\bigskip

%DIF >  \noindent \textbf{Figure \ref{Fig:relative_EI}:}

\DIFadd{We further tested the contributions of elaboration and innovation at different scales by examining the consequences of elaboration and innovation for the observed divergence of bird beak shapes.
Specifically, we calculated the distance to centroid of avian beak morphospace for each species (Fig. \ref{Fig:phylogeny}).
We then used phylogenetic generalised least squares (PGLS; \mbox{%DIFAUXCMD
\cite{phylom}}\hskip0pt%DIFAUXCMD
) to model distance to centroid as a function of $elaboration_{species}$, $innovation_{species}$ and $elaboration_{clade}$ and $innovation_{clade}$.
We ran the PGLS analysis using both single and multiple predictors.
Our analyses indicate that the majority of variation (r$^2$ = 0.888; Table \ref{Tab:pgls}) in beak shape centroid distance can be explained by a combination of $elaboration_{species}$ and $innovation_{species}$.
When modeled as single predictor models, variation explained is lower for $elaboration_{species}$ than $innovation_{species}$ and $innovation_{species}$ has a steeper slope indicating that although elaboration is overall the dominant model of divergence, innovation leads to greater exploration of morphospace (Table \ref{Tab:pgls}).
In contrast, the combination of $elaboration_{clade}$ and $innovation_{clade}$ alone explains only $>$0.01\% of the total variation in beak shape divergence (Table \ref{Tab:pgls}).
However, $innovation_{clade}$ is nonetheless an important contributor to total beak shape space.
We reach this conclusion because models with interactions between species and clade metrics provide by far the best model fit overall and removal of terms indicate that the most important interactions are between $innovation_{clade}$  and $innovation_{species}$ followed by $innovation_{clade}$ and $elaboration_{species}$.
The interaction terms show that the relationship between species metrics ($elaboration_{species}$, $innovation_{clade}$) and distance to centroid becomes steeper as $innovation_{clade}$ increases (Table \ref{Tab:pgls}).
Overall, the models suggest that although most species evolve via elaboration at the macroevolutionary scale, expansions of beak morphospace are also driven by megaevolutionary scale reorientations of trait space. %DIF > @L_PGLS_results
}

\begin{landscape}
\begin{table}[ht]
\centering
\small
   \begin{tabular}{rrllllllllrr}
     \hline
    \DIFaddFL{Model }& \DIFaddFL{$E_{species}$ }& \DIFaddFL{$I_{species}$ }& \DIFaddFL{$E_{clade}$ }& \DIFaddFL{$I_{clade}$ }& \DIFaddFL{$E_{species}$ x $I_{clade}$ }& \DIFaddFL{$E_{species}$ x $E_{clade}$ }& \DIFaddFL{$I_{species}$ x $I_{clade}$ }& \DIFaddFL{$I_{species}$ x $E_{clade}$ }& \DIFaddFL{$\Delta$AIC }& \DIFaddFL{Adj. $r^2$ }\\ 
     \hline
   \DIFaddFL{1 }& \DIFaddFL{0.04*** }& \DIFaddFL{-0.139*** }& \DIFaddFL{0.002 }& \DIFaddFL{-0.002 }& \DIFaddFL{0.064*** }& \DIFaddFL{0.045*** }& \DIFaddFL{0.341*** }& \DIFaddFL{0.093*** }& \DIFaddFL{0.00 }& \DIFaddFL{0.92 }\\ 
   \DIFaddFL{2 }& \DIFaddFL{0.257*** }& \DIFaddFL{0.169*** }& \DIFaddFL{0.01 }& \DIFaddFL{0.001 }& \DIFaddFL{0.045*** }&  & \DIFaddFL{0.312*** }& \DIFaddFL{0.034*** }& \DIFaddFL{-691.48 }& \DIFaddFL{0.92 }\\ 
   \DIFaddFL{3 }& \DIFaddFL{0.108*** }& \DIFaddFL{0.236*** }& \DIFaddFL{0.012 }& \DIFaddFL{-0.002 }& \DIFaddFL{0.055*** }& \DIFaddFL{0.034*** }& \DIFaddFL{0.343*** }&  & \DIFaddFL{-284.74 }& \DIFaddFL{0.92 }\\ 
   \DIFaddFL{4 }& \DIFaddFL{0.254*** }& \DIFaddFL{0.104*** }& \DIFaddFL{0.002 }& \DIFaddFL{0.004 }&  & \DIFaddFL{0.027*** }& \DIFaddFL{0.311*** }& \DIFaddFL{0.052*** }& \DIFaddFL{-877.85 }& \DIFaddFL{0.92 }\\ 
   \DIFaddFL{5 }& \DIFaddFL{0.1*** }& \DIFaddFL{0.571*** }& \DIFaddFL{-0.018* }& \DIFaddFL{0.02*** }& \DIFaddFL{0.054*** }& \DIFaddFL{0.036*** }&  & \DIFaddFL{0.096*** }& \DIFaddFL{-1498.36 }& \DIFaddFL{0.91 }\\ 
   \DIFaddFL{6 }& \DIFaddFL{0.364*** }& \DIFaddFL{0.973*** }& \DIFaddFL{-0.007 }& \DIFaddFL{0.023*** }&  &  &  &  & \DIFaddFL{-2230.10 }& \DIFaddFL{0.90 }\\ 
   \DIFaddFL{7 }& \DIFaddFL{0.364*** }& \DIFaddFL{0.972*** }&  &  &  &  &  &  & \DIFaddFL{-2280.08 }& \DIFaddFL{0.90 }\\ 
   \DIFaddFL{8 }&  &  & \DIFaddFL{-0.025 }& \DIFaddFL{0.011 }&  &  &  &  & \DIFaddFL{-22270.55 }& \DIFaddFL{-0.00 }\\ 
   \DIFaddFL{9 }& \DIFaddFL{0.286*** }&  &  &  &  &  &  &  & \DIFaddFL{-18986.63 }& \DIFaddFL{0.32 }\\ 
   \DIFaddFL{10 }&  & \DIFaddFL{0.798*** }&  &  &  &  &  &  & \DIFaddFL{-17714.99 }& \DIFaddFL{0.41 }\\ 
   \DIFaddFL{11 }&  &  & \DIFaddFL{-0.008 }&  &  &  &  &  & \DIFaddFL{-22269.34 }& \DIFaddFL{-0.00 }\\ 
   \DIFaddFL{12 }&  &  &  & \DIFaddFL{0.002 }&  &  &  &  & \DIFaddFL{-22269.51 }& \DIFaddFL{-0.00 }\\ 
      \hline
   \end{tabular}
\caption{\DIFaddFL{Elaboration and innovation as predictors of beak shape distance from centroid.
The table shows parameter estimates of predictors from 12 different phylogenetic generalised least squares (PGLS) models with $\Delta$AIC showing relative model fit following removal of terms. Models 1-5 are multiple regressions including all variables and one or more interaction terms; models 6-8 are are multiple regressions including two or more variables and no interaction terms; models 9-12 are single predictor models. Values in the table are based on PGLS in which Pagel’s $\lambda$ was fixed at 1 (i.e. assuming a Brownian motion model of evolution). Fixing $\lambda$ is necessary to allow model comparison with AIC. We also fitted the same models fixing $\lambda$ to 0.727 (the lowest value of $\lambda$ from any individual model) and relative model fit and interpretation of parameters was unaffected. E = elaboration; I = innovation; AIC = Akaike Information Criterion; NS = not significant; - indicates that the term was not included within that model.}}
\label{Tab:pgls}
\end{table}
\end{landscape}

\DIFadd{Taken together, these results suggest that }\DIFaddend rather than a paradox\DIFaddbegin \DIFadd{, }\DIFaddend the reorientation of trait space can instead arise as a result of \DIFdelbegin \DIFdel{small innovations that arise in a common direction among closely-related species}\DIFdelend \DIFaddbegin \DIFadd{species-level innovations within clades arising along common directions in phenotypic space}\DIFaddend .
This idea is similar in concept to multiple adaptive peak models where species evolve in a heterogeneous adaptive landscape and share similar responses to selection pressures within adaptive zones \cite{hansen1997stabilizing}.
This implies that gradual directional evolution \cite{pagel2022general}, rather than exceptional megaevolutionary jumps \cite{venditti2011multiple,cooney2017mega}, may be sufficient to explain diversity in avian beak morphology.
\DIFaddbegin \DIFadd{In other words, populations can always change morphologically within the shapespace with minimal amounts of innovation over long periods of time. %DIF > @L_shapespacetravel
}\DIFaddend 

\DIFdelbegin %DIFDELCMD < \begin{figure}[!htbp]
%DIFDELCMD < \centering
%DIFDELCMD <    \includegraphics[width=0.9\textwidth]{Figures/relative_EI_gg_style.pdf}
%DIFDELCMD < %%%
%DIFDELCMD < \caption{%
{%DIFAUXCMD
\DIFdelFL{.}}
%DIFAUXCMD
%DIFDELCMD < \label{Fig:relative_EI}
%DIFDELCMD < \end{figure}
%DIFDELCMD < 

%DIFDELCMD < \bigskip
%DIFDELCMD < 

%DIFDELCMD < \noindent %%%
\textbf{\DIFdel{Figure \ref{Fig:relative_EI}:}} %DIFAUXCMD
\DIFdel{Comparisons of relative elaboration and innovation among clades (illustrated in figure \ref{Fig:ellipses}) and among species (figure \ref{Fig:phylogeny}).
a) the elaboration (blue) for each species in each 4000 posterior samples compared to the elaboration for each of the 35 clades (yellow).
The scores for each clade are scaled by the maximum score within the clade (corresponding to the scaled ellipses in figure \ref{Fig:ellipses}) and the scores for species are scaled by the maximum species elaboration score.
This allows us to compare elaboration in species and clades despite their different sample sizes.
We use the Bhattacharryya coefficient (BC) to quantify the overlap of the area under the curve for species and clades. 
For elaboration the overlap is BC = 0.217, indicating no distinguishable dissimilarity in the amount of elaboration across the posterior samples between species or clades. 
b) the same as a) but using innovation score. 
For innovation the overlap has BC = 0.039, which indicates a clear difference in the amount of innovation across the posterior samples between species and clades. }%DIFDELCMD < 

%DIFDELCMD < %%%
%DIF <  Paragraph 8: conclusion
\DIFdelend \subsection{Conclusions}
\DIFdelbegin %DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelend Our results show that although at the \DIFdelbegin \DIFdel{species-level }\DIFdelend \DIFaddbegin \DIFadd{macroevolutionary scale }\DIFaddend most bird beaks are elaborating along a \DIFdelbegin \DIFdel{megaevolutionary }\DIFdelend \DIFaddbegin \DIFadd{class-wide or sub-clade phylogenetic }\DIFaddend major axis of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation, at the \DIFdelbegin \DIFdel{macroevolutionary level }\DIFdelend \DIFaddbegin \DIFadd{megaevolutionary scale, }\DIFaddend innovation away from the \DIFdelbegin \DIFdel{global }\DIFdelend \DIFaddbegin \DIFadd{class-wide }\DIFaddend phylogenetic major axis of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation is much more common.
This nested structure of elaboration at a lower (species) taxonomic level and innovation at a higher (clade) taxonomic levels could thus explain the diversity of bird beaks we observe today.
However, individual species-level variation in the past could also have led to a shift in a clade's line of least evolutionary resistance through different evolutionary routes on a dynamic adaptive landscape.
Taken together, our results suggest that the signature of evolutionary reorientations in deep-time (\DIFdelbegin \DIFdel{innovation}\DIFdelend \DIFaddbegin \DIFadd{$innovation_{clade}$}\DIFaddend ), coupled with \DIFdelbegin \DIFdel{elaboration at the species-level}\DIFdelend \DIFaddbegin \DIFadd{$elaboration_{species}$}\DIFaddend , is a robust explanation of the massive diversity of bird beaks we observe today and is consistent with recent suggestions \DIFdelbegin \DIFdel{that }\DIFdelend \DIFaddbegin \DIFadd{from univariate analysis that observation of }\DIFaddend apparently abrupt phenotypic shifts \DIFdelbegin \DIFdel{observed in univariate analyses }\DIFdelend can be explained by gradual Darwinian processes \cite{pagel2022general, BurinWhales,goswami2022}.
Indeed, the \DIFdelbegin \DIFdel{global }\DIFdelend \DIFaddbegin \DIFadd{class-wide }\DIFaddend phylogenetic major axis of phenotypic variation is an emergent property of reorientation of trait space among clades that requires no special evolutionary process: megaevolutionary patterns appear to be an inevitable outcome of evolution on a shifting adaptive landscape.

\section{\DIFaddbegin \DIFadd{Materials and }\DIFaddend Methods}

\textbf{Beak shape data.} We used the dataset from \cite{cooney2017mega,hughes2022global,chira2020signature} which consists of four landmarks and three curves of 25 semi-landmarks each, taken from 3D scans of the beaks of 8748 bird species.
The landmark data were aligned with Procrustes superimposition using the R package \texttt{Morpho} \cite{Rcore,Morpho} and we removed the size component and controlled for symmetry (see \cite{cooney2017mega,hughes2022global,chira2020signature} for detailed descriptions of data collection and processing).
We ordinated the superimposed landmarks using a principal components analysis (PCA) and selected the eight first dimensions of the resulting PCA.
These eight dimensions contained at least 95\% of variance in each super-order, and 98.7\% of the variance in the whole dataset.
We used a threshold of \DIFdelbegin \DIFdel{$>95$}\DIFdelend \DIFaddbegin \DIFadd{at least 95}\DIFaddend \% of variance explained for each super-order because although the overall distribution of the variance on each PC axis in the whole trait-space is decreasing (i.e. $\sigma$ PC1 \DIFdelbegin \DIFdel{$<$ }\DIFdelend \DIFaddbegin \DIFadd{$>$ }\DIFaddend $\sigma$ PC2 \DIFdelbegin \DIFdel{$<$ }\DIFdelend \DIFaddbegin \DIFadd{$>$ }\DIFaddend $\sigma$ PC3), this is not always true for each super-order.
For example, in the Mirandornithes, PC2, then PC4, and then PC1 contained the most variance (Fig.
\ref{Fig:ellipses} and \DIFdelbegin \DIFdel{S}\DIFdelend \ref{Fig:axes_variance}).
This protocol for selecting the number of PC axes ensures that the resulting trait-space contains enough variance for each super-order (see Fig. \DIFdelbegin \DIFdel{S}\DIFdelend \ref{Fig:axes_variance}).
This procedure resulted in an $8748 \times 8$ matrix that contains at least 95\% of the variance in all observed living bird beaks (hereafter, the shapespace). %DIF > @L_shapespace_description

\textbf{Phylogenetic data.} We used a random subsample of 1000 trees from the avian tree distribution of \cite{jetz2012global} for the analyses and \DIFdelbegin \DIFdel{the }\DIFdelend one randomly selected tree for the figures (Fig. \ref{Fig:phylogeny}).
We pruned these trees to contain only the 8748 species present in our beak shapespace.

\textbf{Phylogenetic GLMM analysis.} We ran a Bayesian phylogenetic generalised linear mixed model (pGLMM) using the \texttt{MCMCglmm} and \texttt{ape} R packages \cite{MCMCglmm, ape} with the shapespace data as a multidimensional response variable, the error in the dataset as residuals, and each clade's phylogeny as a nested random term, i.e., a model of:

\begin{equation}
\text{shapespace} \mathtt{\sim} \text{residuals(shapespace)} + \text{random(phylogeny)}
\end{equation}

This model allows us to estimate \DIFdelbegin \DIFdel{a phenotypic variance-covariance matrix }\DIFdelend \DIFaddbegin \DIFadd{an evolutionary rate matrix, here defined as a matrix containing the rate of among-species divergence in species traits that explicitly incorporates phylogeny (}\textbf{\DIFadd{R}} \DIFadd{matrix of \mbox{%DIFAUXCMD
\cite{Houle2017} }\hskip0pt%DIFAUXCMD
and }\textbf{\DIFadd{B}} \DIFadd{matrix of \mbox{%DIFAUXCMD
\cite{ Machado2020}}\hskip0pt%DIFAUXCMD
)) }\DIFaddend for each clade and the phylogeny as a whole, and one for the residuals in the shapespace itself \cite{robinson2013quantifying}.
These \DIFdelbegin \DIFdel{variance-covariance }\DIFdelend \DIFaddbegin \DIFadd{evolutionary rate }\DIFaddend matrices were then used in a base projection-rejection analysis to measure the elaboration and innovation scores (see below).
We ran two separate nested models.
The first model used one random term for the \DIFdelbegin \DIFdel{global }\DIFdelend \DIFaddbegin \DIFadd{class-wide }\DIFaddend phylogeny and 35 nested random terms, one for each super-order and order containing more than 15 species in our dataset (Fig.
\ref{Fig:ellipses}).
This resulted in a multidimensional GLMM with a $8748 \times 8$ response variables, one \DIFdelbegin \DIFdel{global }\DIFdelend \DIFaddbegin \DIFadd{class-wide }\DIFaddend residual term and 36 random terms.
The second model was fitted to a subset of the dataset containing only the 5229 Passeriformes species with 30 nested random terms, one for each sub-order and families containing more than 15 species in our dataset (Fig.
\DIFdelbegin \DIFdel{S}\DIFdelend \ref{fig_ellipses_passeriformes}.
This resulted in a multidimensional GLMM with a $5229 \times 8$ response variables, one \DIFdelbegin \DIFdel{global }\DIFdelend \DIFaddbegin \DIFadd{order-wide }\DIFaddend residual term and 30 random terms.
\DIFdelbegin %DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelend To account for phylogenetic uncertainty we ran the model using different tree topologies from the tree distribution in \cite{jetz2012global}.
Because of the very large size of our model and dataset, however, running the full model on multiple trees was computationally unfeasible (it would require 2 CPU years and 4TB of RAM for each of the models, i.e. the \DIFdelbegin \DIFdel{global }\DIFdelend \DIFaddbegin \DIFadd{class-wide }\DIFaddend phylogeny and the Passeriformes models).
Instead, we developed and implemented a ''\DIFdelbegin \DIFdel{mini chains}\DIFdelend \DIFaddbegin \DIFadd{mini-chains}\DIFaddend '' MCMCglmm method using the R package \texttt{mcmcmglmmm} \cite{mcmcmcglmmm}.

\textbf{mcmcmcglmmm.} This method runs relatively short Monte-Carlo Markov chains (MCMC) across a large number of different phylogenetic trees and concatenates the resulting short posteriors into one larger posterior distribution that contains the phylogenetic uncertainty information.
We performed the method using the following protocol (see Fig. \DIFdelbegin \DIFdel{S}\DIFdelend \ref{Fig:mcmcmcglmm}).
First we ran three independent MCMCglmm chains\DIFdelbegin \DIFdel{(}\DIFdelend \DIFaddbegin \DIFadd{, }\DIFaddend hereafter the parameterisation chains\DIFdelbegin \DIFdel{) for 50k }\DIFdelend \DIFaddbegin \DIFadd{, for 50,000 }\DIFaddend generations with the trait data and the model described above on the consensus tree from the tree distribution, along with flat priors with a belief parameter of 2\%, i.e. with a very low weight on the priors\DIFdelbegin \DIFdel{;
}\DIFdelend \DIFaddbegin \DIFadd{.
}\DIFaddend Next we extracted the following parameters from the parameterisation chains: a) the conservative burnin average time across the three chains, defined as the highest number of iterations required to first reach the median posterior likelihood value multiplied by 1.1; and b) the mini-chains priors, defined as the median posterior model values from the parameterisation chains with a belief parameter of 5\%.
We then ran 400 mini chains in parallel with the estimated burnin time and priors to run 10 samples past the burnin.
This resulted in 10 exploitable posterior samples for each tree.
We used 400 trees because that was the number of models required to reach an effective sample size (ESS) of at least 200 for all the estimated parameters (see Fig. \DIFdelbegin \DIFdel{S}\DIFdelend \ref{Fig:model_ess_all_birds} and S\ref{Fig:model_ess_passeriformes}).
Using this approach, we reduced the computational time to around 45 CPU hours and 9GB of RAM per mini-chain - a 400 fold improvement.
The total analysis took around three CPU years using the shARC cluster from the University of Sheffield.
Code to reproduce the procedure is available in the \href{https://raw.rawgit.net/TGuillerme/mcmcmcglmmm/main/inst/MCMCglmm_mini_chains.html}{\texttt{mcmcmcglmmm} vignette} \cite{mcmcmcglmmm}.

\textbf{Elaboration/innovation scores using projection/rejection analyses.} We used the distributions of the 4000 posterior \DIFdelbegin \DIFdel{variance-covariance }\DIFdelend \DIFaddbegin \DIFadd{evolutionary rate }\DIFaddend matrices to run the projection/rejection analyses to obtain elaboration and innovation scores for bird beaks.
We \DIFdelbegin \DIFdel{can use multilinear }\DIFdelend \DIFaddbegin \DIFadd{used linear }\DIFaddend algebra to interpret elaboration and innovation \textit{sensu} \cite{endler2005animal} by using the major axis of the \DIFdelbegin \DIFdel{variance-covariance matrices as the }\DIFdelend \DIFaddbegin \DIFadd{evolutionary rate matrices (referred to throughout the text as the phylogenetic major axis of beak variation) as the }\DIFaddend ''line of least resistance'', or, specifically here, the line of elaboration for each random term (corresponding to the elaboration axis for each clade)\DIFdelbegin \DIFdel{and project }\DIFdelend \DIFaddbegin \DIFadd{.
We projected }\DIFaddend either species or another \DIFaddbegin \DIFadd{phylogenetic }\DIFaddend major axis of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation onto that clade.
\DIFdelbegin \DIFdel{In brief we }\DIFdelend \DIFaddbegin \DIFadd{We }\DIFaddend can then interpret where a species or a clade falls on the \DIFaddbegin \DIFadd{phylogenetic }\DIFaddend major axis of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation of reference as its \DIFdelbegin \DIFdel{elaboration }\DIFdelend \DIFaddbegin \DIFadd{$elaboration_{species}$ }\DIFaddend score, and how far a species is from that \DIFaddbegin \DIFadd{phylogenetic }\DIFaddend major axis of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation as its \DIFdelbegin \DIFdel{innovation }\DIFdelend \DIFaddbegin \DIFadd{$innovation_{species}$ }\DIFaddend score.
We calculated the elaboration and innovation scores using two main approaches: 1) by clades (i.e.
to measure the \DIFdelbegin \DIFdel{elaboration}\DIFdelend \DIFaddbegin \DIFadd{$elaboration_{clade}$}\DIFaddend /\DIFdelbegin \DIFdel{innovation among clades}\DIFdelend \DIFaddbegin \DIFadd{$innovation_{clade}$ at the megaevolutionary scale}\DIFaddend )where we calculated the projections of the \DIFdelbegin \DIFdel{major axes of }\DIFdelend \DIFaddbegin \DIFadd{phylogenetic major axis of beak variation of }\DIFaddend each clade onto the \DIFdelbegin \DIFdel{global }\DIFdelend \DIFaddbegin \DIFadd{class-wide }\DIFaddend phylogenetic major axis of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation (Fig. \ref{Fig:ellipses}); and 2) by species (i.e.
to measure the \DIFdelbegin \DIFdel{elaboration}\DIFdelend \DIFaddbegin \DIFadd{$elaboration_{species}$}\DIFaddend /\DIFdelbegin \DIFdel{innovation within clades}\DIFdelend \DIFaddbegin \DIFadd{$innovation_{species}$ at the macroevolutionary scale}\DIFaddend ) where we calculated the projections of each species onto a) the \DIFdelbegin \DIFdel{global }\DIFdelend \DIFaddbegin \DIFadd{class-wide }\DIFaddend phylogenetic major axis of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation and b) the \DIFaddbegin \DIFadd{phylogenetic }\DIFaddend major axis of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation of their \DIFdelbegin \DIFdel{specific }\DIFdelend \DIFaddbegin \DIFadd{parent }\DIFaddend clade (Fig. \ref{Fig:phylogeny}).
To make the results easier to interpret, we centered and scaled \DIFdelbegin \DIFdel{them onto the centre of the }\DIFdelend \DIFaddbegin \DIFadd{elaboration and innovation scores onto the center of the phylogenetic }\DIFaddend major axis of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation of interest and made the elaboration values absolute.
In this way, we can interpret an elaboration or innovation score within the 0-1 range to be non exceptional\DIFdelbegin \DIFdel{(}\DIFdelend \DIFaddbegin \DIFadd{, }\DIFaddend i.e within the 95\% confidence interval of the \DIFdelbegin \DIFdel{variance-covariance matrix)}\DIFdelend \DIFaddbegin \DIFadd{evolutionary rate matrix}\DIFaddend .
The full mathematical procedure is described in detail in the supplementary materials (section \ref{supp_projection}\DIFdelbegin \DIFdel{).
}\DIFdelend \DIFaddbegin \DIFadd{;  note that the procedure described is applied to each individual evolutionary rate matrix for each term in the model, e.g. for each 4000 posterior variance matrices individually for each $elaboration_{species}$, $innovation_{species}$, $elaboration_{clade}$ or $innovation_{clade}$ score). %DIF > @L_individual_matrices).
}\DIFaddend Code to reproduce the analyses is available \DIFdelbegin \DIFdel{at }\DIFdelend \DIFaddbegin \DIFadd{in }\DIFaddend \href{https://raw.rawgit.net/TGuillerme/dispRity/master/inst/vignettes/Projection_analysis.html}{this \texttt{dispRity} vignette}%DIF <  TODO: update link to release when releasing
\DIFdelbegin \DIFdel{and implemented in the }\texttt{\DIFdel{dispRity}} %DIFAUXCMD
\DIFdel{package \mbox{%DIFAUXCMD
\cite{dispRity}}\hskip0pt%DIFAUXCMD
}\DIFdelend .

\textbf{Clade orthogonality.} We measured the \DIFaddbegin \DIFadd{amount of }\DIFaddend orthogonality of each random term (i.e.
clade) in our models compared to their parent clade and their parent's parent clade.
For example \DIFdelbegin \DIFdel{, the variance-covariance matrices of the }\DIFdelend \DIFaddbegin \DIFadd{for Columbiformes, we measured the amount of orthogonality in the evolutionary ratematrices of (i) the }\DIFaddend Columbimorphae random terms against the phylogeny random terms, \DIFaddbegin \DIFadd{(ii) }\DIFaddend the Columbiformes random terms against the Columbimorphae random terms (parent) and \DIFdelbegin \DIFdel{the }\DIFdelend \DIFaddbegin \DIFadd{(iii) the Columbiformes random terms the }\DIFaddend whole phylogeny random terms (parent's parent).
To measure \DIFaddbegin \DIFadd{the amount of }\DIFaddend orthogonality we calculated the angles between the \DIFdelbegin \DIFdel{major axes of phenotypic }\DIFdelend \DIFaddbegin \DIFadd{phylogenetic major axis of beak }\DIFaddend variation of the focal clade and their parent clade for each posterior \DIFdelbegin \DIFdel{variance-covariance matrix as well as }\DIFdelend \DIFaddbegin \DIFadd{evolutionary rate matrix and }\DIFaddend within each clade, by randomly comparing \DIFaddbegin \DIFadd{phylogenetic }\DIFaddend major axes of \DIFdelbegin \DIFdel{phenotypic }\DIFdelend \DIFaddbegin \DIFadd{beak }\DIFaddend variation within a clade.
We converted each angle measurement into \DIFdelbegin \DIFdel{an }\DIFdelend \DIFaddbegin \DIFadd{a degree of }\DIFaddend orthogonality metric where 0 corresponds to flat angles ($0^\circ$ or  $180^\circ$) and 1 to right angles ($90^\circ$ or $270^\circ$).
We then measured the posterior probability of the angles within a clade being different to the angles among clades.
These results are available in Fig. \ref{Fig:orthogonality}.
\DIFaddbegin \DIFadd{Note that we did not use the random skewers method here \mbox{%DIFAUXCMD
\cite{cheverud2007research} }\hskip0pt%DIFAUXCMD
because we were interested in measuring the amount of orthogonality between pairs of evolutionary rate matrices rather than simply measuring whether they were different (but see supplementary materials Fig. S13 for random skewers correlation results). %DIF > @L_noskewers
}\DIFaddend 

\DIFdelbegin %DIFDELCMD < \bibliographystyle{naturemag}
%DIFDELCMD < \bibliography{references}
%DIFDELCMD < %%%
\DIFdelend \DIFaddbegin \textbf{\DIFadd{Phylogenetic generalised least squares analysis.}} %DIF > @L_pgls_methods
\DIFadd{We ran phylogenetic generalised least squares models (PGLS) to test the effects of elaboration and innovation on beak shape divergence.
Beak shape divergence was calculated as the Euclidean distance of each species beak shape from the centroid of beak shape of all birds.
We then fitted a series of nested models with distance as the response and $elaboration_{species}$, $innovation_{species}$, $elaboration_{clade}$, and $innovation_{clade}$ as predictors.
The most complex model included all two-way interaction terms:
}\DIFaddend 

\DIFdelbegin \textbf{\DIFdel{Acknowledgements.}}
%DIFAUXCMD
\DIFdel{We acknowledge IT Services at The University of Sheffield for the provision of the High Performance Computing Service.
}\DIFdelend \DIFaddbegin \begin{equation}
\DIFadd{\text{distance} \mathtt{\sim} (elaboration_{species} + innovation_{species}) \times (elaboration_{clade} + innovation_{clade})
}\end{equation}

\DIFadd{We then sequentially removed terms to examine effects on model fit.
We initially estimated Pagel's $\lambda$ \mbox{%DIFAUXCMD
\cite{pagel1997inferring} }\hskip0pt%DIFAUXCMD
for all models.
However, model comparison based on AIC scores can be misleading when estimating $\lambda$ (i.e. model differences can arise because of different $\lambda$ fits, rather than due to effects of the parameters of interest).
We therefore repeated all models a further two times, once fixing $\lambda$ to 1 (equivalent to assuming Brownian motion model of evolution) and once fixing $\lambda$ to the lowest value estimated from any of the models.
The results were qualitatively similar regardless of $\lambda$ and we report only models based on pure Brownian motion in the main text.
In addition, we estimated Pagel’s $\lambda$ for $elaboration_{species}$ and $innovation_{species}$ as univariate traits.
All models were fitted using the phylolm library using a single phylogeny \mbox{%DIFAUXCMD
\cite{phylom}}\hskip0pt%DIFAUXCMD
.
}


\section*{\DIFadd{Reproducibility and repeatability}}
\DIFadd{The raw and processed data used in this analysis is available from }\href{https://figshare.com/articles/dataset/Innovation_and_elaboration_on_the_avian_tree_of_life/20480355}{\DIFadd{figshare}} \DIFadd{\mbox{%DIFAUXCMD
\cite{fighsaredata}}\hskip0pt%DIFAUXCMD
.
The code used to reproduce the figures and tables is available from }\href{https://github.com/TGuillerme/elaboration_exploration_bird_beaks}{\DIFadd{github}} \DIFadd{\mbox{%DIFAUXCMD
\cite{githubrepo}}\hskip0pt%DIFAUXCMD
.
}

\section*{\DIFadd{Acknowledgements}}
\DIFaddend We thank M. Adams, H. van Grouw \DIFdelbegin \DIFdel{, }\DIFdelend \DIFaddbegin \DIFadd{and }\DIFaddend R. Prys-Jones \DIFdelbegin \DIFdel{and and Zo\"{e} K. Varley }\DIFdelend from the Bird Group at the NHM, Tring \DIFdelbegin \DIFdel{.
}\DIFdelend \DIFaddbegin \DIFadd{and H. McGhie at the Manchester Museum for providing access to and expertise in the collections.
}\DIFaddend We are indebted to the volunteer citizen scientists at \DIFdelbegin \DIFdel{http://www.markmybird.org }\DIFdelend \DIFaddbegin \url{http://www.markmybird.org} \DIFaddend for helping to build the database of bird \DIFdelbegin \DIFdel{beak }\DIFdelend \DIFaddbegin \DIFadd{bill }\DIFaddend shape and contribute to our understanding of avian evolution. 
\DIFdelbegin \textbf{\DIFdel{Funding.}}
%DIFAUXCMD
\DIFdel{This work was funded by the European Research Council (615709 Project ‘ToLERates’), UKRI-NERC Grant NE/T000139/1 and a Royal Society University Research Fellowship (URF R 180006 to G.
H.T.).
}\textbf{\DIFdel{Author contributions.}}
%DIFAUXCMD
\DIFdelend \DIFaddbegin \DIFadd{We thank Masahito Tusboi and three other anonymous reviewers for their suggestions that greatly helped improving the manuscript.
}

\section*{\DIFadd{Author contributions}} \DIFaddend G.H.T., A.P.B., N.C., and T. G. designed the study. J.A.B., C.R.C., E.C.H., \DIFaddbegin \DIFadd{Z.K.V., }\DIFaddend and G.H.T. %DIF <  and Z.K.V.
curated the data. T.G. analysed the data. All authors contributed to the writing of the manuscript.
\DIFdelbegin \textbf{\DIFdel{Competing interests.}}
%DIFAUXCMD
\DIFdelend \DIFaddbegin 

\section*{\DIFadd{Competing interests}} \DIFaddend We declare no competing interests.
\DIFdelbegin \textbf{\DIFdel{Data an materials availability.}}
%DIFAUXCMD
\DIFdel{The raw and processed data used in this analysis is available from }\href{https://figshare.shef.ac.uk/account/articles/21526146}{\DIFdel{figshare}} %DIFAUXCMD
\DIFdel{\mbox{%DIFAUXCMD
\cite{fighsaredata}}\hskip0pt%DIFAUXCMD
.
The code used to reproduce the figures and tables is available from }\href{https://github.com/TGuillerme/elaboration_exploration_bird_beaks}{\DIFdel{GitHub}} %DIFAUXCMD
\DIFdel{\mbox{%DIFAUXCMD
\cite{githubrepo}}\hskip0pt%DIFAUXCMD
}\DIFdelend \DIFaddbegin 

\section*{\DIFadd{Funding}}
\DIFadd{This work was funded by UKRI-NERC Grant NE/T000139/1 and a Royal Society University Research Fellowship (URF R 180006 to GHT)}\DIFaddend .


%DIF <  TODO: Add list of supplementary materials
%DIF <  All this comes after the references
%DIF <  Figures and captions after this
\DIFaddbegin \bibliographystyle{naturemag}
\bibliography{references}
\DIFaddend 


\end{document}